--- a/PKGBUILD	2018-03-12 17:32:57.049007339 +0100
+++ b/PKGBUILD	2018-03-12 17:45:14.783595573 +0100
@@ -14,26 +14,28 @@ url='https://github.com/ivmai/libatomic_
 depends=(glibc)
 license=(GPL2 MIT)
 options=(!docs)
-source=($url/releases/download/v$pkgver/${pkgname}-${pkgver}.tar.gz)
-sha256sums=('5b823d5a685dd70caeef8fc50da7d763ba7f6167fe746abca7762e2835b3dd4e')
+makedepends=(git)
+source=(git+https://github.com/ivmai/libatomic_ops.git)
+sha256sums=('SKIP')
 
 prepare() {
-  cd $pkgname-$pkgver
+  cd $pkgname
 }
 
 build() {
-  cd $pkgname-$pkgver
+  cd $pkgname
+  autoreconf --install
   ./configure --prefix=/usr --disable-static --enable-shared
   make
 }
 
 check() {
-  cd $pkgname-$pkgver
+  cd $pkgname
   make check
 }
 
 package() {
-  cd $pkgname-$pkgver
+  cd $pkgname
   make DESTDIR="$pkgdir" install
   install -Dm644 doc/LICENSING.txt "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
 }
