diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-06-05 14:01:54.301835807 +0200
+++ b/PKGBUILD	2018-06-05 14:03:35.098503480 +0200
@@ -5,7 +5,7 @@
 # Contributor: Link Dupont <link@subpop.net>
 
 pkgbase=dbus
-pkgname=(dbus dbus-docs)
+pkgname=(dbus)
 pkgver=1.12.8
 pkgrel=1
 pkgdesc="Freedesktop.org message bus system"
@@ -13,7 +13,7 @@ url="https://wiki.freedesktop.org/www/So
 arch=(x86_64)
 license=(GPL custom)
 depends=(libsystemd expat)
-makedepends=(systemd xmlto docbook-xsl python yelp-tools doxygen git autoconf-archive graphviz)
+makedepends=(systemd docbook-xsl python git autoconf-archive)
 _commit=f8902fa1c840e3fab807016e6a00647b6ef9dd41  # tags/dbus-1.12.8^0
 source=("git+https://anongit.freedesktop.org/git/dbus/dbus#commit=$_commit"
         'dbus.sysusers')
