--- a/PKGBUILD	2018-03-17 08:59:41.689090995 +0100
+++ b/PKGBUILD	2018-03-17 09:05:37.822823499 +0100
@@ -13,7 +13,6 @@ url='http://packages.debian.org/fakeroot
 groups=(base-devel)
 install=fakeroot.install
 depends=(glibc filesystem sed util-linux sh)
-makedepends=(po4a)
 checkdepends=(sharutils)
 source=(http://ftp.debian.org/debian/pool/main/f/$pkgname/${pkgname}_${pkgver}.orig.tar.bz2
         silence-dlerror.patch)
@@ -36,9 +34,6 @@ build() {
     --with-ipc=sysv
 
   make
-
-  cd doc
-  po4a -k 0 --rm-backups --variable 'srcdir=../doc/' po4a/po4a.cfg
 }
 
 check() {
