diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-06-03 19:37:49.390312733 +0200
+++ b/PKGBUILD	2018-06-03 19:52:17.116988058 +0200
@@ -11,7 +11,6 @@ url="http://git-scm.com/"
 license=('GPL2')
 depends=('curl' 'expat>=2.0' 'perl>=5.14.0' 'perl-error' 'perl-mailtools'
          'openssl' 'pcre2' 'grep' 'shadow')
-makedepends=('python2' 'emacs' 'libgnome-keyring' 'xmlto' 'asciidoc')
 optdepends=('tk: gitk and git gui'
             'perl-libwww: git svn'
             'perl-term-readkey: git svn'
@@ -55,12 +54,9 @@ build() {
 
   make \
     "${make_options[@]}" \
-    all doc
+    all
 
-  make -C contrib/emacs prefix=/usr
-  make -C contrib/credential/gnome-keyring
-  make -C contrib/credential/libsecret
-  make -C contrib/subtree prefix=/usr gitexecdir=/usr/lib/git-core all doc
+  make -C contrib/subtree prefix=/usr gitexecdir=/usr/lib/git-core all
   make -C contrib/mw-to-git prefix=/usr all
   make -C contrib/diff-highlight prefix=/usr
 }
@@ -92,7 +88,7 @@ package() {
   make \
     "${make_options[@]}" \
     DESTDIR="$pkgdir" \
-    install install-doc
+    install
 
   # bash completion
   mkdir -p "$pkgdir"/usr/share/bash-completion/completions/
@@ -100,18 +96,8 @@ package() {
   # fancy git prompt
   mkdir -p "$pkgdir"/usr/share/git/
   install -m 0644 ./contrib/completion/git-prompt.sh "$pkgdir"/usr/share/git/git-prompt.sh
-  # emacs
-  make -C contrib/emacs prefix=/usr DESTDIR="$pkgdir" install
-  # gnome credentials helper
-  install -m 0755 contrib/credential/gnome-keyring/git-credential-gnome-keyring \
-      "$pkgdir"/usr/lib/git-core/git-credential-gnome-keyring
-  make -C contrib/credential/gnome-keyring clean
-  # libsecret credentials helper
-  install -m 0755 contrib/credential/libsecret/git-credential-libsecret \
-      "$pkgdir"/usr/lib/git-core/git-credential-libsecret
-  make -C contrib/credential/libsecret clean
   # subtree installation
-  make -C contrib/subtree prefix=/usr gitexecdir=/usr/lib/git-core DESTDIR="$pkgdir" install install-doc
+  make -C contrib/subtree prefix=/usr gitexecdir=/usr/lib/git-core DESTDIR="$pkgdir" install
   # mediawiki installation
   make -C contrib/mw-to-git prefix=/usr gitexecdir=/usr/lib/git-core DESTDIR="$pkgdir" install
   mv "$pkgdir"/Git/Mediawiki.pm "$pkgdir"/usr/share/perl5/vendor_perl/Git/ && rmdir "$pkgdir"/Git # TODO: fix install path!
