diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-05-24 12:07:38.664755661 +0200
+++ b/PKGBUILD	2018-05-24 12:11:10.631424442 +0200
@@ -6,13 +6,13 @@
 # NOTE: valgrind requires rebuilt with each major glibc version
 
 pkgbase=glibc
-pkgname=(glibc lib32-glibc)
+pkgname=(glibc)
+[ "x@MULTILIB@" != "xenable" ] || pkgname+=(lib32-glibc)
 pkgver=2.27
 pkgrel=3
 arch=(x86_64)
 url='http://www.gnu.org/software/libc'
 license=(GPL LGPL)
-makedepends=(git gd lib32-gcc-libs)
 options=(!strip staticlibs)
 _commit=23158b08a0908f381459f273a984c6fd328363cb
 #source=(git+https://sourceware.org/git/glibc.git#commit=$_commit
@@ -53,7 +53,7 @@ build() {
       --enable-add-ons
       --enable-bind-now
       --enable-lock-elision
-      --enable-multi-arch
+      --@MULTILIB@-multi-arch
       --enable-stack-protector=strong
       --enable-stackguard-randomization
       --enable-static-pie
@@ -87,6 +87,8 @@ build() {
   echo "CXX += -D_FORTIFY_SOURCE=2" >> configparms
   make
 
+  [ "x@MULTILIB@" == "xenable" ] || return 0
+
   cd "$srcdir/lib32-glibc-build"
   export CC="gcc -m32 -mstackrealign"
   export CXX="g++ -m32 -mstackrealign"
