diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-06-03 10:29:39.543317896 +0200
+++ b/PKGBUILD	2018-06-03 10:31:08.406652115 +0200
@@ -4,14 +4,14 @@
 # Contributor: Sarah Hay <sarah@archlinux.org>
 
 pkgbase=gpgme
-pkgname=(gpgme qgpgme python-gpgme python2-gpgme)
+pkgname=(gpgme python-gpgme python2-gpgme)
 pkgver=1.11.1
 pkgrel=1
 pkgdesc="A C wrapper library for GnuPG"
 arch=('x86_64')
 url="http://www.gnupg.org/related_software/gpgme/"
 license=('LGPL')
-makedepends=('libgpg-error' 'gnupg' 'qt5-base' 'python' 'python2' 'swig')
+makedepends=('libgpg-error' 'gnupg' 'python' 'python2')
 source=("https://www.gnupg.org/ftp/gcrypt/${pkgbase}/${pkgbase}-${pkgver}.tar.bz2"{,.sig})
 sha256sums=('2d1b111774d2e3dd26dcd7c251819ce4ef774ec5e566251eb9308fa7542fbd6f'
             'SKIP')
@@ -40,9 +40,9 @@ package_gpgme() {
   make DESTDIR="${pkgdir}" install
 
   # split qgpgme
-  rm -r "${pkgdir}"/usr/include/{qgpgme,QGpgME}/
-  rm -r "${pkgdir}"/usr/lib/{cmake/QGpgme/,libqgpgme.*}
-  rm -r "${pkgdir}"/usr/lib/python*
+  rm -rf "${pkgdir}"/usr/include/{qgpgme,QGpgME}/
+  rm -rf "${pkgdir}"/usr/lib/{cmake/QGpgme/,libqgpgme.*}
+  rm -rf "${pkgdir}"/usr/lib/python*
 }
 
 package_qgpgme() {
