diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-06-06 14:34:13.659383914 +0200
+++ b/PKGBUILD	2018-06-06 14:49:49.749393254 +0200
@@ -8,7 +8,7 @@
 pkgname=libtool
 pkgver=2.4.6+40+g6ca5e224
 pkgrel=6
-_gccver=8.1.0
+_gccver=7.3.1
 pkgdesc='A generic library support script'
 arch=(x86_64)
 url='http://www.gnu.org/software/libtool'
@@ -36,7 +36,7 @@ prepare() {
 
 build() {
   cd libtool
-  ./configure --prefix=/usr lt_cv_sys_lib_dlsearch_path_spec="/usr/lib /usr/lib32"
+  ./configure --prefix=/usr lt_cv_sys_lib_dlsearch_path_spec="/usr/lib"
   make
 }
 
