diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-06-05 18:38:14.958667912 +0200
+++ b/PKGBUILD	2018-06-05 18:39:30.538668672 +0200
@@ -10,7 +10,7 @@ url='https://www.open-mpi.org'
 arch=('x86_64')
 license=('custom:OpenMPI')
 depends=('libltdl' 'hwloc' 'openssh' 'zlib' 'libnl')
-makedepends=('inetutils' 'valgrind' 'gcc-fortran')
+makedepends=('gcc-fortran')
 optdepends=('gcc-fortran: fortran support')
 options=('staticlibs')
 source=(https://www.open-mpi.org/software/ompi/v${pkgver%.*}/downloads/${pkgname}-${pkgver}.tar.bz2)
@@ -28,8 +28,6 @@ build() {
     --enable-smp-locks \
     --enable-builtin-atomics \
     --enable-mpi-cxx \
-    --with-valgrind \
-    --enable-memchecker \
     --enable-pretty-print-stacktrace \
     --without-slurm \
     --with-hwloc=/usr \
