--- a/PKGBUILD	2018-03-16 16:15:27.773981022 +0100
+++ b/PKGBUILD	2018-03-16 16:17:14.902033427 +0100
@@ -10,7 +10,6 @@ pkgdesc='Free version of the SSH connect
 url='https://www.openssh.com/portable.html'
 license=('custom:BSD')
 arch=('x86_64')
-makedepends=('linux-headers')
 depends=('krb5' 'openssl' 'libedit' 'ldns')
 optdepends=('xorg-xauth: X11 forwarding'
             'x11-ssh-askpass: input passphrase in X')
