diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-06-05 02:21:58.501416775 +0200
+++ b/PKGBUILD	2018-06-05 08:23:26.441633177 +0200
@@ -9,7 +9,7 @@ arch=(x86_64)
 url="https://p11-glue.freedesktop.org"
 license=(BSD)
 depends=(glibc libtasn1 libffi)
-makedepends=(gtk-doc git)
+makedepends=(git)
 _commit=3dd5810143e51dabdc58069e55b09a950349fa08  # tags/0.23.12^0
 source=("git+https://github.com/p11-glue/p11-kit#commit=$_commit"
         0001-Build-and-install-libnssckbi-p11-kit.so.patch)
@@ -41,7 +41,7 @@ build() {
     --sysconfdir=/etc \
     --localstatedir=/var \
     --libexecdir=/usr/lib \
-    --enable-doc \
+    --disable-doc \
     --with-module-path=/usr/lib/pkcs11 \
     --with-trust-paths=/etc/ca-certificates/trust-source:/usr/share/ca-certificates/trust-source
   make
