diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-06-03 10:32:25.883319558 +0200
+++ b/PKGBUILD	2018-06-03 10:32:59.146653221 +0200
@@ -13,7 +13,6 @@ arch=('x86_64')
 license=('PSF')
 url="http://www.python.org/"
 depends=('bzip2' 'expat' 'gdbm' 'libffi' 'libnsl' 'openssl' 'sqlite' 'zlib')
-makedepends=('tk' 'bluez-libs')
 checkdepends=('gdb' 'file' 'xorg-server-xvfb')
 optdepends=('tk: for IDLE'
             'python2-setuptools'
@@ -67,7 +66,7 @@ build() {
   ./configure --prefix=/usr \
               --enable-shared \
               --with-threads \
-              --enable-optimizations \
+              --disable-optimizations \
               --with-lto \
               --enable-ipv6 \
               --enable-unicode=ucs4 \
