diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-06-06 05:21:02.252386067 +0200
+++ b/PKGBUILD	2018-06-06 06:15:14.332418516 +0200
@@ -85,6 +85,10 @@ sha512sums=('SKIP'
 _backports=(
   # nspawn: wait for network namespace creation before interface setup (#8633)
   '7511655807e90aa33ea7b71991401a79ec36bb41'
+  # basic/macros: rename noreturn into _noreturn_ (#8456)
+  '848e863acc51ecfb0f3955c498874588201d9130'
+  # core: don't include libmount.h in a header file (#8580)
+  '227b8a762fea1458547be2cdf0e6e4aac0079730'
 )
 
 _reverts=(
@@ -134,7 +138,9 @@ prepare() {
   git remote add -f upstream ../systemd
   git checkout "$_commit"
 
-  _validate_tag || return
+  # temporary hack due to upstream build issues
+  # https://tests.reproducible-builds.org/archlinux/core/systemd/build1.log
+  #_validate_tag || return
 
   local c
   for c in "${_backports[@]}"; do
@@ -261,7 +267,7 @@ package_systemd() {
   rm -rv "$pkgdir"/etc/systemd/system/*
 
   # we'll create this on installation
-  rmdir "$pkgdir"/var/log/journal/remote
+  rm -rf "$pkgdir"/var/log/journal/remote
 
   # executable (symlinks) shipped with systemd-sysvcompat
   rm "$pkgdir"/usr/bin/{halt,init,poweroff,reboot,runlevel,shutdown,telinit}
