diff -rupN a/usr/bin/librechroot b/usr/bin/librechroot
--- a/usr/bin/librechroot	2018-01-03 23:05:52.000000000 +0100
+++ b/usr/bin/librechroot	2018-02-26 09:14:42.813251799 +0100
@@ -69,32 +69,36 @@ hack_arch_nspawn_flags() {
 		CARCH="$(uname -m)"
 	fi
 
-	if [[ "$CARCH" == armv7h ]] && ! setarch armv7l /bin/true 2>/dev/null; then
-		# We're running an ARM chroot on a non-ARM processor
+	case $CARCH in
+		armv7h) _setarch=armv7l ; _qemuarch=arm    ;;
+		*)      _setarch=$CARCH ; _qemuarch=$CARCH ;;
+	esac
+	if ! setarch $_setarch /bin/true 2>/dev/null; then
+		# We're running a cross-arch chroot
 
 		# Make sure that qemu-static is set up with binfmt_misc
 		if [[ -z $(grep -l -xF \
-			     -e 'interpreter /usr/bin/qemu-arm-static' \
-			     -r -- /proc/sys/fs/binfmt_misc 2>/dev/null \
-			   | xargs -r grep -xF 'enabled') ]]
+			      -e "interpreter /usr/bin/qemu-$_qemuarch-static" \
+			      -r -- /proc/sys/fs/binfmt_misc 2>/dev/null \
+			   | xargs -r grep -xF 'enabled') ]]
 		then
-			error 'Cannot cross-compile for ARM on x86'
-			plain 'This requires a binfmt_misc entry for qemu-arm-static.'
+			error "Cannot cross-compile for $CARCH on $(uname -m)"
+			plain "This requires a binfmt_misc entry for qemu-$_qemuarch-static."
 			prose 'Such a binfmt_misc entry is provided by the %s
-			       package.  If you have it installed, but still see
-			       this message, you may need to restart %s.' \
+			       package.  If you have it installed, but still see
+			       this message, you may need to restart %s.' \
 				qemu-user-static-binfmt systemd-binfmt.service
 			return $EXIT_NOTINSTALLED
 		fi
 
 		# Let qemu/binfmt_misc do its thing
-		arch_nspawn_flags+=(-f /usr/bin/qemu-arm-static -s)
+		arch_nspawn_flags+=(-f /usr/bin/qemu-$_qemuarch-static -s)
 
 		# The -any packages are built separately for ARM from
 		# x86, so if we use the same CacheDir as the x86 host,
 		# then there will be PGP errors.
-		mkdir -p /var/cache/pacman/pkg-arm
-		arch_nspawn_flags+=(-c /var/cache/pacman/pkg-arm)
+		mkdir -p /var/cache/pacman/pkg-$CARCH
+		arch_nspawn_flags+=(-c /var/cache/pacman/pkg-$CARCH)
 	fi
 }
 
