diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-04-04 23:14:14.847738317 +0200
+++ b/PKGBUILD	2018-04-09 13:39:05.411065213 +0200
@@ -11,14 +11,12 @@ url='https://github.com/lathiat/avahi'
 license=(LGPL)
 arch=(x86_64)
 depends=(expat libdaemon glib2 libcap gdbm dbus)
-makedepends=(git qt4 pygtk mono intltool gtk-sharp-2 gobject-introspection gtk3
+makedepends=(git pygtk intltool gtk-sharp-2 gobject-introspection gtk3
              xmltoman python-dbus python-gobject doxygen graphviz)
 optdepends=('gtk3: avahi-discover, avahi-discover-standalone, bshell, bssh, bvnc'
             'gtk2: gtk2 bindings'
-            'qt4: qt4 bindings'
             'pygtk: avahi-bookmarks'
             'python2-twisted: avahi-bookmarks'
-            'mono: mono bindings'
             'nss-mdns: NSS support for mDNS'
             'python-gobject: avahi-discover'
             'python-dbus: avahi-discover')
@@ -43,7 +41,7 @@ prepare() {
 
 build() {
   cd $pkgname
-  export MOC_QT4=/usr/bin/moc-qt4 PYTHON=/usr/bin/python3
+  export PYTHON=/usr/bin/python3
 
   ./configure \
     --prefix=/usr \
