diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-06-06 17:17:39.352815084 +0200
+++ b/PKGBUILD	2018-06-06 17:17:47.479481833 +0200
@@ -38,6 +38,8 @@ build() {
 
+   _bindir="bin.linux"
    [[ "${CARCH}" = "x86_64" ]] && _bindir="bin.linuxx86_64"
+   [[ "${CARCH}" = "riscv64" ]] && _bindir="bin.linux"
+   [[ "${CARCH}" = "powerpc64le" ]] && _bindir="bin.linuxppc"
    install -Dm755 tools/build/src/engine/$_bindir/b2 "${_stagedir}"/bin/b2
 
    # Support for OpenMPI
