diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-04-09 07:49:37.470052499 +0200
+++ b/PKGBUILD	2018-04-09 07:49:37.470052499 +0200
@@ -10,7 +10,7 @@ arch=(x86_64)
 url="https://www.freedesktop.org/software/colord"
 license=(GPL2)
 depends=(lcms2 libgusb polkit sqlite systemd dconf dbus libgudev shared-mime-info)
-makedepends=(gobject-introspection vala sane bash-completion argyllcms git docbook-utils
+makedepends=(gobject-introspection bash-completion argyllcms git docbook-utils
              docbook-sgml perl-sgmls meson gtk-doc)
 optdepends=('sane: scanner support'
             'argyllcms: color profiling')
@@ -36,8 +36,8 @@ prepare() {
 build() {
   arch-meson $pkgname build \
     -Denable-libcolordcompat=true \
-    -Denable-sane=true \
-    -Denable-vala=true \
+    -Denable-sane=false \
+    -Denable-vala=false \
     -Denable-print-profiles=true \
     -Dwith-daemon-user=colord
   ninja -C build
