diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-06-15 02:49:04.173387228 +0200
+++ b/PKGBUILD	2018-06-15 07:43:24.360230106 +0200
@@ -28,15 +28,16 @@ prepare() {
 build() {
   cd ${pkgbase}-Release_${pkgver//./_}/build
 
+  # FIXME: ironically, building docs breaks on risc-v.
   cmake .. \
     -DCMAKE_INSTALL_PREFIX:PATH=/usr \
     -DDOC_INSTALL_DIR:PATH=share/doc/doxygen \
     -DPYTHON_EXECUTABLE:FILE=/usr/bin/python2 \
-    -Dbuild_doc:BOOL=ON \
-    -Dbuild_wizard:BOOL=ON
+    -Dbuild_doc:BOOL=OFF \
+    -Dbuild_wizard:BOOL=OFF
 
   make
-  make docs
+  #make docs
 }
 
 check(){
