diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-06-09 11:09:28.288514268 +0200
+++ b/PKGBUILD	2018-06-09 11:26:01.755190846 +0200
@@ -21,7 +21,7 @@ depends=('alsa-lib' 'bzip2' 'fontconfig'
          'xz' 'zlib'
          'libass.so' 'libbluray.so' 'libfreetype.so' 'libva-drm.so' 'libva.so'
          'libva-x11.so' 'libvidstab.so' 'libvorbisenc.so' 'libvorbis.so'
-         'libvpx.so' 'libx264.so' 'libx265.so' 'libxvidcore.so')
+         'libvpx.so' 'libxvidcore.so')
 makedepends=('ffnvcodec-headers' 'ladspa' 'yasm')
 optdepends=('ladspa: LADSPA filters')
 provides=('libavcodec.so' 'libavdevice.so' 'libavfilter.so' 'libavformat.so'
@@ -69,12 +69,9 @@ build() {
     --enable-libvorbis \
     --enable-libvpx \
     --enable-libwebp \
-    --enable-libx264 \
-    --enable-libx265 \
     --enable-libxcb \
     --enable-libxml2 \
     --enable-libxvid \
-    --enable-nvenc \
     --enable-omx \
     --enable-shared \
     --enable-version3
