diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-04-11 06:20:43.633180499 +0200
+++ b/PKGBUILD	2018-04-11 08:24:00.586479436 +0200
@@ -1,6 +1,8 @@
 # $Id$
 # Maintainer: Jan de Groot <jgc@archlinux.org>
 
+# FIXME: doctools segfaults when building docs.
+
 pkgname=fontconfig
 pkgver=2.13.0+10+g58f5285
 pkgrel=1
@@ -9,7 +11,7 @@ arch=(x86_64)
 url="https://www.freedesktop.org/wiki/Software/fontconfig/"
 license=(custom)
 depends=(expat freetype2)
-makedepends=(git autoconf-archive gperf python-lxml python-six docbook-utils docbook-sgml
+makedepends=(git autoconf-archive gperf python-lxml python-six
              perl-sgmls 'texlive-formatsextra>=2017' lynx)
 install=fontconfig.install
 _commit=58f52853d5689e897525a5926c1a222340d3f404  # master
