diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-06-08 10:57:17.527644887 +0200
+++ b/PKGBUILD	2018-06-08 14:28:36.927771400 +0200
@@ -12,7 +12,7 @@ depends=(libxml2 polkit libldap dbus-gli
 makedepends=(intltool gtk-doc gobject-introspection git gnome-common)
 install=gconf.install
 _commit=0780809731c8ab1c364202b1900d3df106b28626  # master
-source=("git+https://git.gnome.org/browse/gconf#commit=$_commit"
+source=("git+https://gitlab.gnome.org/Archive/gconf.git#commit=$_commit"
         01_xml-gettext-domain.patch gconf-reload.patch
         gconf-merge-schema gconfpkg gconf-{install,remove}.hook)
 sha256sums=('SKIP'
