diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-06-11 09:08:06.590165790 +0200
+++ b/PKGBUILD	2018-06-11 09:08:48.736832878 +0200
@@ -11,7 +11,7 @@ arch=(x86_64)
 url='https://community.kde.org/Frameworks'
 license=(LGPL)
 depends=(kwidgetsaddons kconfig)
-makedepends=(extra-cmake-modules qt5-tools clang python-pyqt5 python2-pyqt5 doxygen)
+makedepends=(extra-cmake-modules qt5-tools python-pyqt5 python2-pyqt5 doxygen)
 optdepends=('python-pykf5: to use the Python 3 bindings' 'python2-pykf5: to use the Python 2 bindings')
 groups=(kf5)
 source=("https://download.kde.org/stable/frameworks/${pkgver%.*}/$pkgname-$pkgver.tar.xz"{,.sig})
