diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-06-09 13:07:58.168585209 +0200
+++ b/PKGBUILD	2018-06-09 14:49:45.735312816 +0200
@@ -11,7 +11,7 @@ arch=(x86_64)
 url='https://community.kde.org/Frameworks'
 license=(LGPL)
 depends=(qt5-base shared-mime-info)
-makedepends=(extra-cmake-modules qt5-tools clang python-pyqt5 python2-pyqt5 doxygen)
+makedepends=(extra-cmake-modules qt5-tools python-pyqt5 python2-pyqt5 doxygen)
 optdepends=('python-pykf5: to use the Python 3 bindings' 'python2-pykf5: to use the Python 2 bindings')
 groups=(kf5)
 source=("https://download.kde.org/stable/frameworks/${pkgver%.*}/$pkgname-$pkgver.tar.xz"{,.sig})
