diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-06-11 09:07:30.496832098 +0200
+++ b/PKGBUILD	2018-06-11 09:08:01.836832410 +0200
@@ -11,7 +11,7 @@ arch=(x86_64)
 url='https://community.kde.org/Frameworks'
 license=(LGPL)
 depends=(qt5-x11extras)
-makedepends=(extra-cmake-modules clang python-pyqt5 python2-pyqt5 doxygen qt5-tools)
+makedepends=(extra-cmake-modules python-pyqt5 python2-pyqt5 doxygen qt5-tools)
 optdepends=('python-pykf5: to use the Python 3 bindings' 'python2-pykf5: to use the Python 2 bindings')
 groups=(kf5)
 source=("https://download.kde.org/stable/frameworks/${pkgver%.*}/$pkgname-$pkgver.tar.xz"{,.sig})
