diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-04-07 21:43:46.276686533 +0200
+++ b/PKGBUILD	2018-04-07 21:43:46.276686533 +0200
@@ -14,7 +14,7 @@ optdepends=('networkmanager: NetworkMana
             'python: Python 3.x bindings'
             'glib2: gsettings configuration module'
             'webkit2gtk: PAC proxy support (Webkit2gtk engine)')
-makedepends=('cmake' 'networkmanager' 'python' 'python2' 'perl' 'webkit2gtk')
+makedepends=('cmake' 'networkmanager' 'python' 'python2' 'perl')
 url="http://libproxy.github.io/libproxy/"
 source=(https://github.com/libproxy/libproxy/archive/${pkgver}.tar.gz)
 sha256sums=('18f58b0a0043b6881774187427ead158d310127fc46a1c668ad6d207fb28b4e0')
@@ -33,7 +33,7 @@ build() {
     -DCMAKE_SKIP_RPATH=ON \
     -DPERL_VENDORINSTALL=yes \
     -DCMAKE_BUILD_TYPE=Release \
-    -DWITH_WEBKIT3=ON \
+    -DWITH_WEBKIT3=OFF \
     -DWITH_MOZJS=OFF \
     -DCMAKE_CXX_FLAGS="${CXXFLAGS}" \
     -DCMAKE_C_FLAGS="${CFLAGS}" \
