diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-06-12 06:37:53.950937954 +0200
+++ b/PKGBUILD	2018-06-12 06:44:08.554275025 +0200
@@ -2,7 +2,7 @@
 # Maintainer: Jan de Groot <jgc@archlinux.org>
 
 pkgname=librsvg
-pkgver=2.42.5
+pkgver=2.40.20
 pkgrel=1
 epoch=2
 pkgdesc="SVG rendering library"
@@ -10,35 +10,25 @@ url="https://wiki.gnome.org/Projects/Lib
 arch=(x86_64)
 license=(LGPL)
 depends=(gdk-pixbuf2 pango libcroco)
-makedepends=(intltool gobject-introspection vala python2 gtk-doc git rust)
+makedepends=(intltool gobject-introspection python2 gtk-doc git)
 _commit=497c925593e41c12cec4a55e4365655a29c3fe59  # tags/2.42.5^0
-source=("git+https://gitlab.gnome.org/GNOME/librsvg.git#commit=$_commit")
-sha256sums=('SKIP')
-
-pkgver() {
-  cd $pkgname
-  git describe --tags | sed 's/-/+/g'
-}
-
-prepare() {
-  cd $pkgname
-  NOCONFIGURE=1 ./autogen.sh
-}
+source=(https://ftp.gnome.org/pub/GNOME/sources/librsvg/${pkgver%.*}/librsvg-$pkgver.tar.xz)
+sha256sums=('cff4dd3c3b78bfe99d8fcfad3b8ba1eee3289a0823c0e118d78106be6b84c92b')
 
 build() {
-  cd $pkgname
-  ./configure --prefix=/usr --disable-static --enable-vala --enable-gtk-doc
+  cd $pkgname-$pkgver
+  ./configure --prefix=/usr --disable-static --disable-vala --disable-gtk-doc
   sed -i -e 's/ -shared / -Wl,-O1,--as-needed\0 /g' libtool
   make
 }
 
 check() {
-  cd $pkgname
+  cd $pkgname-$pkgver
   make check
 }
 
 package() {
-  cd $pkgname
+  cd $pkgname-$pkgver
   make DESTDIR="$pkgdir" install
 }
 
