diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-06-11 04:15:24.603323892 +0200
+++ b/PKGBUILD	2018-06-12 16:03:57.157943501 +0200
@@ -12,7 +12,7 @@ url="http://libsigc.sourceforge.net/"
 makedepends=('gcc-libs' 'git' 'mm-common' 'doxygen' 'libxslt' 'graphviz')
 options=('!emptydirs')
 _commit=83f1e2fe7855f85af570b9653903d2c426d67e72
-source=("git://git.gnome.org/libsigcplusplus#commit=$_commit")
+source=("git+https://github.com/GNOME/libsigcplusplus.git#commit=$_commit")
 sha256sums=('SKIP')
 
 pkgver() {
