diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-06-07 22:58:48.533881418 +0200
+++ b/PKGBUILD	2018-06-07 23:00:00.987215473 +0200
@@ -9,8 +9,6 @@ arch=('x86_64')
 url="https://hwmon.wiki.kernel.org/lm_sensors"
 license=('GPL' 'LGPL')
 depends=('perl')
-makedepends=('rrdtool')
-optdepends=('rrdtool: for logging with sensord')
 backup=('etc/sensors3.conf' 'etc/healthd.conf' 'etc/conf.d/sensord')
 source=($pkgname-$pkgver::https://github.com/groeck/lm-sensors/archive/V${pkgver//\./-}.tar.gz
 	healthd healthd.conf healthd.service sensord.conf)
@@ -34,7 +32,7 @@ build() {
 
 package() {
   cd ${pkgname/_/-}-${pkgver//\./-}
-  make PROG_EXTRA=sensord BUILD_STATIC_LIB=0 \
+  make BUILD_STATIC_LIB=0 \
     PREFIX=/usr SBINDIR=/usr/bin MANDIR=/usr/share/man DESTDIR="${pkgdir}" install
 
   install -D -m755 "${srcdir}/healthd" "${pkgdir}/usr/bin/healthd"
