--- a/PKGBUILD	2018-04-04 01:51:42.340031672 +0200
+++ b/PKGBUILD	2018-04-04 08:33:35.667746275 +0200
@@ -11,7 +11,7 @@ url="https://cgit.freedesktop.org/gstrea
 arch=(x86_64)
 license=(custom)
 depends=(glibc)
-makedepends=(valgrind git gtk-doc meson)
+makedepends=(git gtk-doc meson)
 _commit=31cb4bfc51de81b5c2569abdcff830b83c74499c  # tags/orc-0.4.28^0
 source=("git+https://anongit.freedesktop.org/git/gstreamer/orc#commit=$_commit")
 sha256sums=('SKIP')
