diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-06-06 05:17:51.495717497 +0200
+++ b/PKGBUILD	2018-06-06 05:17:51.499050831 +0200
@@ -24,15 +24,13 @@ check() {
   make test
 }
 
+perlver() { perl -e '$v = $^V->{version}; print $v->[0].".".($v->[1]+$ARGV[0]);' $1; }
+
 package() {
   cd XML-Parser-${pkgver}
   make DESTDIR="${pkgdir}" install
 
-# template start; name=perl-binary-module-dependency; version=1;
 if [[ $(find "$pkgdir/usr/lib/perl5/" -name "*.so") ]]; then
-	_perlver_min=$(perl -e '$v = $^V->{version}; print $v->[0].".".($v->[1]);')
-	_perlver_max=$(perl -e '$v = $^V->{version}; print $v->[0].".".($v->[1]+1);')
-	depends+=("perl>=$_perlver_min" "perl<$_perlver_max")
+	depends+=("perl>=$(perlver)" "perl<$(perlver 1)")
 fi
-# template end;
 }
