diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-04-08 21:19:01.690638474 +0200
+++ b/PKGBUILD	2018-04-08 21:19:01.691638474 +0200
@@ -3,13 +3,13 @@
 # Maintainer: Jan de Groot <jgc@archlinux.org>
 
 pkgbase=poppler
-pkgname=('poppler' 'poppler-glib' 'poppler-qt5')
+pkgname=('poppler' 'poppler-glib')
 pkgver=0.63.0
 pkgrel=1
 arch=(x86_64)
 license=('GPL')
-makedepends=('libjpeg' 'gcc-libs' 'cairo' 'fontconfig' 'openjpeg2' 'gtk3' 'pkgconfig' 'lcms2' 
-             'gobject-introspection' 'icu' 'qt5-base' 'git' 'nss' 'gtk-doc' 'curl' 'poppler-data'
+makedepends=('libjpeg' 'gcc-libs' 'cairo' 'fontconfig' 'openjpeg2' 'pkgconfig' 'lcms2' 
+             'gobject-introspection' 'icu' 'git' 'nss' 'gtk-doc' 'curl' 'poppler-data'
              'cmake' 'python') 
 options=('!emptydirs')
 url="https://poppler.freedesktop.org/"
