diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-04-07 17:13:27.064223747 +0200
+++ b/PKGBUILD	2018-04-07 17:13:27.064223747 +0200
@@ -14,7 +14,7 @@ arch=('x86_64')
 license=('custom')
 url="http://www.python.org/"
 depends=('expat' 'bzip2' 'gdbm' 'openssl' 'libffi' 'zlib')
-makedepends=('tk' 'sqlite' 'valgrind' 'bluez-libs' 'mpdecimal' 'llvm' 'gdb' 'xorg-server-xvfb')
+makedepends=('tk' 'sqlite' 'bluez-libs' 'mpdecimal' 'llvm' 'xorg-server-xvfb')
 optdepends=('python-setuptools'
             'python-pip'
             'sqlite'
@@ -55,7 +55,7 @@ build() {
               --enable-shared \
               --with-threads \
               --with-computed-gotos \
-              --enable-optimizations \
+              --disable-optimizations \
               --with-lto \
               --enable-ipv6 \
               --with-system-expat \
