diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-06-11 18:55:02.357183836 +0200
+++ b/PKGBUILD	2018-06-11 19:01:36.640521104 +0200
@@ -17,13 +17,14 @@ sha256sums=('SKIP')
 prepare() {
   cd "${srcdir}"
   ln -s ${_pkgname} "${_pkgname}-sdk-${pkgver}"
+
+  cd "${srcdir}/${_pkgname}-sdk-${pkgver}"
+  ./update_external_sources.sh
 }
 
 build() {
   cd "${srcdir}/${_pkgname}-sdk-${pkgver}"
 
-  ./update_external_sources.sh
-
   rm -rf build ; mkdir build ; cd build
   cmake -DCMAKE_INSTALL_PREFIX=/usr \
     -DCMAKE_INSTALL_LIBDIR=lib \
