--- a/PKGBUILD	2018-03-19 19:58:13.014101363 +0100
+++ b/PKGBUILD	2018-03-19 20:00:52.026776851 +0100
@@ -14,6 +14,7 @@ source=("http://archlinuxarm.org/builder
         "http://archlinuxarm.org/builder/src/${pkgname}-${pkgver}.tar.gz.sig")
 sha256sums=('1bff06fee35a13afaa4bf6b62a833a157f3e3abcb66bc57e11d30d6d0c4f9351'
             'SKIP')
+validpgpkeys=('68B3537F39A313B3E574D06777193F152BDBE6A6')
 
 package() {
 	cd "${srcdir}/${pkgname}-${pkgver}"
