diff -rupN a/0001-fix-build-with-e2fsprogs-1.44.0.patch b/0001-fix-build-with-e2fsprogs-1.44.0.patch
--- a/0001-fix-build-with-e2fsprogs-1.44.0.patch	1970-01-01 01:00:00.000000000 +0100
+++ b/0001-fix-build-with-e2fsprogs-1.44.0.patch	2018-04-05 10:05:27.065035156 +0200
@@ -0,0 +1,14 @@
+diff --git a/convert/source-ext2.c b/convert/source-ext2.c
+index b1492c78693d..070126ec38ec 100644
+--- a/convert/source-ext2.c
++++ b/convert/source-ext2.c
+@@ -422,8 +422,7 @@ static int ext2_xattr_check_entry(struct ext2_ext_attr_entry *entry,
+ {
+ 	size_t value_size = entry->e_value_size;
+ 
+-	if (entry->e_value_block != 0 || value_size > size ||
+-	    entry->e_value_offs + value_size > size)
++	if (value_size > size || entry->e_value_offs + value_size > size)
+ 		return -EIO;
+ 	return 0;
+ }
diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-04-05 10:05:27.064035156 +0200
+++ b/PKGBUILD	2018-04-05 10:05:27.065035156 +0200
@@ -20,7 +20,8 @@ source=("https://www.kernel.org/pub/linu
         'initcpio-install-btrfs'
         'initcpio-hook-btrfs'
         'btrfs-scrub@.service'
-        'btrfs-scrub@.timer')
+        'btrfs-scrub@.timer'
+        '0001-fix-build-with-e2fsprogs-1.44.0.patch')
 install=btrfs-progs.install
 options=(!staticlibs)
 md5sums=('SKIP'
@@ -28,7 +29,13 @@ md5sums=('SKIP'
          '7241ba3a4286d08da0d50b7176941112'
          'b09688a915a0ec8f40e2f5aacbabc9ad'
          '917b31f4eb90448d6791e17ea0f386c7'
-         '502221c1b47a3bb2c06703d4fb90a0c2')
+         '502221c1b47a3bb2c06703d4fb90a0c2'
+         '66a2b1ec9f0d5e8a34fb9344d6307ee0')
+
+prepare() {
+  cd $pkgname-v$pkgver
+  patch -Np1 -i ../0001-fix-build-with-e2fsprogs-1.44.0.patch
+}
 
 build() {
   cd $pkgname-v$pkgver
