--- a/PKGBUILD	2018-03-12 18:02:55.550559093 +0100
+++ b/PKGBUILD	2018-03-12 18:19:24.997339010 +0100
@@ -10,17 +10,19 @@ arch=('x86_64')
 url="http://www.hboehm.info/gc/"
 license=('GPL')
 depends=('gcc-libs' 'libatomic_ops')
-source=(https://github.com/ivmai/bdwgc/releases/download/v${pkgver}/${pkgname}-${pkgver}.tar.gz)
-sha256sums=('b94c1f2535f98354811ee644dccab6e84a0cf73e477ca03fb5a3758fb1fecd1c')
+makedepends=('git')
+source=(git+https://github.com/ivmai/bdwgc.git)
+sha256sums=('SKIP')
 
 prepare() {
-  cd ${pkgname}-${pkgver}
+  cd bdwgc
+  autoreconf --install
 }
 
 build() {
-  cd ${pkgname}-${pkgver}
+  cd bdwgc
   ./configure --prefix=/usr --enable-cplusplus --disable-static
   sed -i -e 's/ -shared / -Wl,-O1,--as-needed\0/g' libtool
   make
@@ -10,7 +10,7 @@ arch=('x86_64')
 }
 
 package() {
-  cd ${pkgname}-${pkgver}
+  cd bdwgc
   make DESTDIR="${pkgdir}" install
   sed 's|GC_MALLOC 1L|gc 3|g' doc/gc.man |
     install -Dm644 /dev/stdin "${pkgdir}/usr/share/man/man3/gc.3"
