--- a/PKGBUILD	2018-04-04 01:43:36.102077757 +0200
+++ b/PKGBUILD	2018-04-04 08:23:33.783803320 +0200
@@ -10,7 +10,7 @@ url="https://gstreamer.freedesktop.org/"
 arch=(x86_64)
 license=(LGPL)
 depends=(libxml2 glib2 libunwind libcap libelf)
-makedepends=(gtk-doc gobject-introspection autoconf-archive git valgrind bash-completion)
+makedepends=(gtk-doc gobject-introspection autoconf-archive git bash-completion)
 checkdepends=(gmp gsl gtk3)
 _commit=80e0e90d00bb358ca9dc071cf6de010ec5ae2644  # tags/1.14.0^0
 install=gstreamer.install
