diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-04-11 14:52:50.921268248 +0200
+++ b/PKGBUILD	2018-04-11 14:56:52.850245319 +0200
@@ -10,7 +10,7 @@ arch=('i686' 'x86_64')
 url="http://gtkglext.sourceforge.net/"
 license=('LGPL')
 depends=('gtk2' 'glu' 'libxmu' 'pangox-compat')
-makedepends=('mesa')
+makedepends=('mesa' 'python')
 source=("http://downloads.sourceforge.net/sourceforge/gtkglext/${pkgname}-${pkgver}.tar.bz2"
 	'gtk2.20.patch')
 md5sums=('ed7ba24ce06a8630c07f2d0ee5f04ab4'
