diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-04-07 19:17:07.815520427 +0200
+++ b/PKGBUILD	2018-04-07 19:17:07.816520427 +0200
@@ -11,7 +11,6 @@ arch=('x86_64')
 url='https://01.org/linuxmedia/vaapi'
 license=('MIT')
 depends=('glibc' 'libdrm' 'libgl' 'libx11' 'libxext' 'libxfixes' 'wayland')
-makedepends=('mesa')
 optdepends=('libva-vdpau-driver: backend for Nvidia and AMD cards'
             'libva-intel-driver: backend for Intel cards')
 provides=('libva-drm.so' 'libva-glx.so' 'libva-wayland.so' 'libva-x11.so' 'libva.so')
