--- a/PKGBUILD	2018-03-19 20:17:44.647454342 +0100
+++ b/PKGBUILD	2018-03-19 20:25:11.605615751 +0100
@@ -13,7 +13,7 @@ makedepends=('git' 'gmp')
 arch=('any')
 url="https://jxself.org/git/?p=linux-libre-firmware.git;a=summary"
 license=('GPL2' 'GPL3')
-depends=('ath9k-htc-firmware' 'openfwwf')
+depends=('openfwwf')
 provides=('linux-firmware')
 conflicts=('linux-firmware'
            'linux-firmware-git'
@@ -60,6 +60,7 @@ pkgver() {
 
 build() {
   cd "${srcdir}/${pkgname}"
+  MAKEFLAGS=""
   make cis dsp56k isci keyspan_pda #av7110 carl9170fw usbdux
 }
 
