--- a/PKGBUILD	2018-03-26 06:52:22.298062888 +0200
+++ b/PKGBUILD	2018-03-26 06:56:52.174607798 +0200
@@ -20,7 +20,7 @@ sha256sums=('98fe1d3c311156c597cd5cfa767
 prepare() {
   cd "busybox-$pkgver"
 
-  local safeflags="-march=${CARCH/_/-} -mtune=generic -Os -pipe -fno-strict-aliasing"
+  local safeflags="-march=rv64gc -mabi=lp64d -Os -pipe -fno-strict-aliasing"
 
   sed 's|^\(CONFIG_EXTRA_CFLAGS\)=.*|\1="'"$safeflags"'"|' "$srcdir/config" > .config
 }
