diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-04-10 13:59:21.242761216 +0200
+++ b/PKGBUILD	2018-04-10 13:59:21.243761216 +0200
@@ -11,37 +11,35 @@ url="https://wiki.gnome.org/Projects/Val
 arch=(x86_64)
 license=(LGPL)
 depends=(glib2 gtk-doc graphviz ttf-font pkg-config gcc)
-makedepends=(libxslt vala git help2man)
+makedepends=(libxslt git help2man)
 checkdepends=(dbus libx11 gobject-introspection)
 provides=(valadoc)
 conflicts=(valadoc)
 replaces=(valadoc)
 _commit=3a11e70aefae4f2344b6352440e82cab0963f4f7  # tags/0.40.2^0
-source=("git+https://git.gnome.org/browse/vala#commit=$_commit")
-sha256sums=('SKIP')
+source=("git://git.gnome.org/vala-bootstrap"
+        "git+https://git.gnome.org/browse/vala#commit=$_commit")
+sha256sums=('SKIP'
+            'SKIP')
 
-pkgver() {
-  cd $pkgname
-  git describe --tags | sed 's/-/+/g'
-}
+build() {
+  cd $pkgname-bootstrap
+  ./autogen.sh
+  ./configure --prefix=$srcdir/vala-bootstrap --disable-build-from-vala \
+    --disable-vapigen --disable-silent-rules
+  sed -i -e 's/ -shared / -Wl,-O1,--as-needed\0/g' libtool
+  make
+  make install
+  cd ..
 
-prepare() {
   cd $pkgname
+  export VALAC=$srcdir/vala-bootstrap/bin/valac
   NOCONFIGURE=1 ./autogen.sh
-}
-
-build() {
-  cd $pkgname
   ./configure --prefix=/usr
   sed -i -e 's/ -shared / -Wl,-O1,--as-needed\0/g' libtool
   make
 }
 
-check() {
-  cd $pkgname
-  make check
-}
-
 package() {
   cd $pkgname
   make DESTDIR="$pkgdir" install
