--- a/PKGBUILD	2018-03-05 08:54:51.864853816 +0100
+++ b/PKGBUILD	2018-03-05 08:54:51.864853816 +0100
@@ -4,8 +4,8 @@
 # toolchain build order: linux-api-headers->glibc->binutils->gcc->binutils->glibc
 
 pkgname=binutils
-pkgver=2.29.1
-pkgrel=2
+pkgver=2.30
+pkgrel=1
 pkgdesc='A set of programs to assemble and manipulate binary and object files'
 arch=(x86_64)
 url='http://www.gnu.org/software/binutils/'
@@ -22,7 +22,7 @@ options=(staticlibs !distcc !ccache)
 source=(https://ftp.gnu.org/gnu/binutils/binutils-$pkgver.tar.xz{,.sig}
         0001-x86-64_Dont_pass_output_bfd_to_info-callbacks-minfo.patch)
 validpgpkeys=(3A24BC1E8FB409FA9F14371813FCEF89DD9E3C4F)
-md5sums=('acc9cd826edb9954ac7cecb81c727793'
+md5sums=('ffc476dd46c96f932875d1b2e27e929f'
          'SKIP'
          'e4be936139ef46122cb3841881c432b2')
 
@@ -43,6 +43,7 @@ build() {
   cd binutils-build
 
   "$srcdir/binutils-$pkgver/configure" \
+    --host=@CHOST@ --build=@BUILDHOST@ \
     --prefix=/usr \
     --with-lib-path=/usr/lib:/usr/local/lib \
     --with-bugurl=https://bugs.archlinux.org/ \
@@ -56,7 +57,7 @@ build() {
     --with-pic \
     --disable-werror \
     --disable-gdb \
-    --with-system-zlib
+    --with-system-zlib=@SYSROOT@/usr/lib/libz.so
 
   make configure-host
   make tooldir=/usr
