--- a/PKGBUILD	2018-03-02 08:39:58.598726010 +0100
+++ b/PKGBUILD	2018-03-02 08:39:58.596725979 +0100
@@ -5,7 +5,8 @@
 # toolchain build order: linux-api-headers->glibc->binutils->gcc->binutils->glibc
 # NOTE: libtool requires rebuilt with each new gcc version
 
-pkgname=(gcc gcc-libs gcc-fortran gcc-objc gcc-ada gcc-go lib32-gcc-libs)
+pkgname=(gcc gcc-libs gcc-fortran gcc-objc gcc-ada gcc-go)
+[ "x@MULTILIB@" == "disable" ] || pkgname+=(lib32-gcc-libs)
 pkgver=7.3.0
 _majorver=${pkgver:0:1}
 _islver=0.18
@@ -77,7 +78,15 @@ build() {
   CFLAGS=${CFLAGS/-pipe/}
   CXXFLAGS=${CXXFLAGS/-pipe/}
 
-  "$srcdir/gcc/configure" --prefix=/usr \
+  CFLAGS=${CFLAGS/-march=@GCC_MARCH@}
+  CFLAGS=${CFLAGS/-mabi=@GCC_MABI@}
+  CXXFLAGS=${CXXFLAGS/-march=@GCC_MARCH@}
+  CXXFLAGS=${CXXFLAGS/-mabi=@GCC_MABI@}
+
+  sed -i 's/gcc_cv_objdump/ac_cv_prog_OBJDUMP/' $srcdir/gcc/libcc1/configure
+  "$srcdir/gcc/configure" --host=@TARGET@ --build=@BUILDHOST@ --target=@TARGET@ --prefix=/usr \
+      --with-arch=@GCC_MARCH@ --with-api=@GCC_MABI@ \
+      --disable-bootstrap \
       --libdir=/usr/lib \
       --libexecdir=/usr/lib \
       --mandir=/usr/share/man \
@@ -101,7 +110,7 @@ build() {
       --enable-install-libiberty \
       --with-linker-hash-style=gnu \
       --enable-gnu-indirect-function \
-      --enable-multilib \
+      --@MULTILIB@-multilib \
       --disable-werror \
       --enable-checking=release \
       --enable-default-pie \
