--- a/PKGBUILD	2018-03-02 08:04:38.298423124 +0100
+++ b/PKGBUILD	2018-03-02 08:04:38.297423126 +0100
@@ -6,16 +6,17 @@
 # NOTE: valgrind requires rebuilt with each major glibc version
 
 pkgbase=glibc
-pkgname=(glibc lib32-glibc)
-pkgver=2.26
-pkgrel=11
+pkgname=(glibc)
+[ "x@MULTILIB@" != "xenable" ] || pkgname+=(lib32-glibc)
+pkgver=2.27
+pkgrel=1
 arch=(x86_64)
 url='http://www.gnu.org/software/libc'
 license=(GPL LGPL)
 makedepends=(git gd lib32-gcc-libs)
 options=(!strip staticlibs)
 _commit=de51f431ed6226ec68ca76e578f2cbd55b6262cb
-source=(git+https://sourceware.org/git/glibc.git#commit=${_commit}
+source=(git+https://sourceware.org/git/glibc.git#tag=glibc-$pkgver
         locale.gen.txt
         locale-gen
         lib32-glibc.conf
@@ -44,12 +45,13 @@ prepare() {
 build() {
   local _configure_flags=(
       --prefix=/usr
-      --with-headers=/usr/include
+      --host=@TARGET@ --build=@BUILDHOST@
+      --with-headers=@SYSROOT@/usr/include
       --with-bugurl=https://bugs.archlinux.org/
       --enable-add-ons
       --enable-bind-now
       --enable-lock-elision
-      --enable-multi-arch
+      --@MULTILIB@-multi-arch
       --enable-obsolete-nsl
       --enable-obsolete-rpc
       --enable-stack-protector=strong
@@ -84,6 +86,8 @@ build() {
   echo "CXX += -D_FORTIFY_SOURCE=2" >> configparms
   make
 
+  [ "x@MULTILIB@" == "xenable" ] || return 0
+
   cd "$srcdir/lib32-glibc-build"
   export CC="gcc -m32 -mstackrealign"
   export CXX="g++ -m32 -mstackrealign"
@@ -169,17 +173,17 @@ package_glibc() {
 
   if check_option 'debug' n; then
     cd "$pkgdir"
-    strip $STRIP_BINARIES usr/bin/{gencat,getconf,getent,iconv,iconvconfig} \
+    @TARGET@-strip $STRIP_BINARIES usr/bin/{gencat,getconf,getent,iconv,iconvconfig} \
                           usr/bin/{ldconfig,locale,localedef,nscd,makedb} \
                           usr/bin/{pcprofiledump,pldd,rpcgen,sln,sprof} \
                           usr/lib/getconf/*
 
-    strip $STRIP_STATIC usr/lib/lib{anl,BrokenLocale,c{,_nonshared},crypt}.a \
+    @TARGET@-strip $STRIP_STATIC usr/lib/lib{anl,BrokenLocale,c{,_nonshared},crypt}.a \
                         usr/lib/lib{dl,g,ieee,mcheck,nsl,pthread{,_nonshared}}.a \
                         usr/lib/lib{resolv,rpcsvc,rt,util}.a \
-                        usr/lib/lib{m-${pkgver},mvec{,_nonshared}}.a
+                        usr/lib/lib{m-${pkgver},mvec{,_nonshared}}.a || true
 
-    strip $STRIP_SHARED usr/lib/lib{anl,BrokenLocale,cidn,crypt}-${pkgver}.so \
+    @TARGET@-strip $STRIP_SHARED usr/lib/lib{anl,BrokenLocale,cidn,crypt}-${pkgver}.so \
                         usr/lib/libnss_{compat,db,dns,files,hesiod,nis,nisplus}-*.so \
                         usr/lib/lib{dl,m,nsl,resolv,rt,util}-${pkgver}.so \
                         usr/lib/lib{memusage,pcprofile,SegFault}.so \
@@ -211,11 +215,11 @@ package_lib32-glibc() {
 
   if check_option 'debug' n; then
     cd $pkgdir
-    strip $STRIP_BINARIES usr/lib32/getconf/*
-    strip $STRIP_STATIC usr/lib32/lib{anl,BrokenLocale,c{,_nonshared},crypt}.a \
+    @TARGET@-strip $STRIP_BINARIES usr/lib32/getconf/*
+    @TARGET@-strip $STRIP_STATIC usr/lib32/lib{anl,BrokenLocale,c{,_nonshared},crypt}.a \
                         usr/lib32/lib{dl,g,ieee,mcheck,nsl,pthread{,_nonshared}}.a \
-                        usr/lib32/lib{resolv,rpcsvc,rt,util,m}.a
-    strip $STRIP_SHARED usr/lib32/lib{anl,BrokenLocale,cidn,crypt}-${pkgver}.so \
+                        usr/lib32/lib{resolv,rpcsvc,rt,util,m}.a || true
+    @TARGET@-strip $STRIP_SHARED usr/lib32/lib{anl,BrokenLocale,cidn,crypt}-${pkgver}.so \
                         usr/lib32/libnss_{compat,db,dns,files,hesiod,nis,nisplus}-*.so \
                         usr/lib32/lib{dl,m,nsl,resolv,rt,util}-${pkgver}.so \
                         usr/lib32/lib{memusage,pcprofile,SegFault}.so \
