--- a/PKGBUILD	2018-02-21 21:54:28.027494092 +0100
+++ b/PKGBUILD	2018-02-21 22:13:00.992257732 +0100
@@ -2,7 +2,7 @@
 # Maintainer: Jan de Groot <jgc@archlinux.org>
 
 pkgbase=glib2
-pkgname=(glib2 glib2-docs)
+pkgname=(glib2)
 pkgver=2.56.0+7+g66948ae23
 pkgrel=1
 pkgdesc="Low level core library"
@@ -43,13 +43,18 @@ build() {
   check_option debug n && debug=yes
 
   cd glib
+  cat > config.cache << EOF
+glib_cv_stack_grows=no
+glib_cv_uscore=no
+EOF
   ./configure \
+    --host=@CHOST@ --build=@BUILDHOST@ \
+    --cache-file=config.cache \
     --prefix=/usr \
     --libdir=/usr/lib \
     --sysconfdir=/etc \
     --with-pcre=system \
     --enable-debug=$debug \
-    --enable-gtk-doc \
     --disable-fam
   sed -i -e 's/ -shared / -Wl,-O1,--as-needed\0/g' libtool
   make
@@ -63,7 +68,6 @@ check() {
 package_glib2() {
   cd glib
   make DESTDIR="$pkgdir" install
-  mv "$pkgdir/usr/share/gtk-doc" "$srcdir"
 
   install -Dt "$pkgdir/usr/share/libalpm/hooks" -m644 ../*.hook
 }
