--- a/PKGBUILD	2018-02-22 11:35:45.855515252 +0100
+++ b/PKGBUILD	2018-02-22 11:37:33.121645532 +0100
@@ -4,7 +4,7 @@
 # Contributor: Sarah Hay <sarah@archlinux.org>
 
 pkgbase=gpgme
-pkgname=(gpgme qgpgme python-gpgme python2-gpgme)
+pkgname=(gpgme)
 pkgver=1.10.0
 pkgrel=2
 pkgdesc="A C wrapper library for GnuPG"
@@ -20,6 +20,7 @@ validpgpkeys=('D8692123C4065DEA5E0F3AB52
 build() {
   cd ${pkgbase}-${pkgver}
   ./configure --prefix=/usr --disable-fd-passing --disable-static \
+              --host=@CHOST@ --build=@BUILDHOST@ \
               --disable-gpgsm-test
   make
 }
@@ -37,9 +38,9 @@ package_gpgme() {
   make DESTDIR="${pkgdir}" install
 
   # split qgpgme
-  rm -r "${pkgdir}"/usr/include/{qgpgme,QGpgME}/
-  rm -r "${pkgdir}"/usr/lib/{cmake/QGpgme/,libqgpgme.*}
-  rm -r "${pkgdir}"/usr/lib/python*
+  rm -r "${pkgdir}"/usr/include/{qgpgme,QGpgME}/ || true
+  rm -r "${pkgdir}"/usr/lib/{cmake/QGpgme/,libqgpgme.*} || true
+  rm -r "${pkgdir}"/usr/lib/python* || true
 }
 
 package_qgpgme() {
