--- a/PKGBUILD	2018-03-04 10:52:27.843093733 +0100
+++ b/PKGBUILD	2018-03-04 10:52:27.843093733 +0100
@@ -14,26 +14,27 @@ url='https://github.com/ivmai/libatomic_
 depends=(glibc)
 license=(GPL2 MIT)
 options=(!docs)
-source=($url/releases/download/v$pkgver/${pkgname}-${pkgver}.tar.gz)
-sha256sums=('219724edad3d580d4d37b22e1d7cb52f0006d282d26a9b8681b560a625142ee6')
+source=(git+https://github.com/ivmai/libatomic_ops.git)
+sha256sums=('SKIP')
 
 prepare() {
-  cd $pkgname-$pkgver
+  cd $pkgname
 }
 
 build() {
-  cd $pkgname-$pkgver
-  ./configure --prefix=/usr --disable-static --enable-shared
+  cd $pkgname
+  autoreconf --install
+  ./configure --host=@TARGET@ --build=@BUILDHOST@ --prefix=/usr --disable-static --enable-shared
   make
 }
 
 check() {
-  cd $pkgname-$pkgver
+  cd $pkgname
   make check
 }
 
 package() {
-  cd $pkgname-$pkgver
+  cd $pkgname
   make DESTDIR="$pkgdir" install
   install -Dm644 doc/LICENSING.txt "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
 }
