--- a/PKGBUILD	2018-03-05 09:37:25.042537462 +0100
+++ b/PKGBUILD	2018-03-05 09:37:25.042537462 +0100
@@ -70,6 +70,12 @@ build() {
   export CFLAGS="$CFLAGS -fPIC"
   cd cyrus-sasl-$pkgver
 
+  sed -i '/^AC_PROG_CC/aAC_PROG_CC_FOR_BUILD' configure.in
+  cat >> include/Makefile.am << 'EOF'
+makemd5: $(makemd5_SOURCES)
+	$(CC_FOR_BUILD) -o $@ $^
+EOF
+
   rm -f config/config.guess config/config.sub 
   rm -f config/ltconfig config/ltmain.sh config/libtool.m4
   rm -fr autom4te.cache
@@ -91,6 +97,7 @@ build() {
   popd
 
   ./configure --prefix=/usr \
+      --host=@TARGET@ --build=@BUILDHOST@ \
       --sbin=/usr/bin \
       --mandir=/usr/share/man \
       --infodir=/usr/share/info \
@@ -112,18 +119,19 @@ build() {
       --enable-ntlm \
       --disable-passdss \
       --enable-sql \
-      --with-mysql=/usr \
-      --with-pgsql=/usr/lib \
+      --without-mysql \
+      --without-pgsql \
       --with-sqlite3=/usr/lib \
-      --enable-ldapdb \
+      --disable-ldapdb \
       --disable-macos-framework \
       --with-pam \
       --with-saslauthd=/var/run/saslauthd \
-      --with-ldap \
+      --without-ldap \
       --with-dblib=gdbm \
       --with-configdir=/etc/sasl2:/etc/sasl:/usr/lib/sasl2 \
       --sysconfdir=/etc \
       --with-devrandom=/dev/urandom
+  sed -i 's#/usr/include##' Makefile */Makefile
   make
 }
 
