--- a/PKGBUILD	2018-02-16 08:07:12.100498831 +0100
+++ b/PKGBUILD	2018-02-16 08:10:02.378659844 +0100
@@ -18,7 +18,8 @@ validpgpkeys=('1F42418905D8206AA754CCDC2
 
 build() {
   cd ${pkgname}-${pkgver}
-  ./configure --prefix=/usr \
+  @CONFIG_SUB@ > build-aux/config.sub
+  ./configure --host=@TARGET@ --build=@BUILDHOST@ --prefix=/usr \
       --with-packager=Archlinux \
       --with-packager-bug-reports="http://bugs.archlinux.org/" \
       --with-packager-version=${pkgver}-${pkgrel}
