--- a/PKGBUILD	2018-02-20 18:48:24.326408636 +0100
+++ b/PKGBUILD	2018-02-20 18:54:14.080912013 +0100
@@ -39,9 +39,11 @@ build() {
 		openssltarget='linux-elf'
 		optflags=''
 	fi
+	openssltarget='linux-elf'
 
 	# mark stack as non-executable: http://bugs.archlinux.org/task/12434
 	./Configure --prefix=/usr --openssldir=/etc/ssl --libdir=lib \
+		--cross-compile-prefix=@CHOST@- no-asm \
 		shared no-ssl3-method ${optflags} \
 		"${openssltarget}" \
 		"-Wa,--noexecstack ${CPPFLAGS} ${CFLAGS} ${LDFLAGS}"
