--- a/PKGBUILD	2018-03-29 08:40:33.725839233 +0200
+++ b/PKGBUILD	2018-03-29 08:47:05.303802120 +0200
@@ -24,7 +24,7 @@ makedepends=('acl' 'cryptsetup' 'docbook
              'meson' 'libseccomp' 'pcre2')
 makedepends_i686=('gnu-efi-libs')
 makedepends_x86_64=('gnu-efi-libs')
-options=('strip')
+depends=('libutil-linux' 'pcre2' 'libidn' 'libseccomp' 'kmod')
 validpgpkeys=('63CDA1E5D3FC22B998D20DD6327F26951A015CC4'  # Lennart Poettering <lennart@poettering.net>
               '5C251B5FC54EB2F80F407AAAC54CA336CFEB557E') # Zbigniew Jędrzejewski-Szmek <zbyszek@in.waw.pl>
 source=('git+https://github.com/systemd/systemd-stable'
@@ -85,6 +85,8 @@ sha512sums=('SKIP'
 _backports=(
   # core: do not free heap-allocated strings (#8391) (FS#57741)
   '5cbaad2f6795088db56063d20695c6444595822f'
+  # basic/macros: rename noreturn into _noreturn_ (#8456)
+  '848e863acc51ecfb0f3955c498874588201d9130'
 )
 
 _reverts=(
@@ -163,10 +165,30 @@ build() {
     CXXFLAGS+=" -fno-lto"
   fi
 
+  cat > cross-file << EOF
+[binaries]
+c = '@CHOST@-gcc'
+cpp = '@CHOST@-g++'
+ar = '@CHOST@-ar'
+strip = '@CHOST@-strip'
+pkgconfig = 'pkg-config'
+[host_machine]
+system = 'linux'
+cpu_family = '@CARCH@'
+cpu = '@GCC_MARCH@'
+endian = 'little'
+EOF
+
   local meson_options=(
     -Daudit=false
     -Dima=false
     -Dlz4=true
+    -Dlibcryptsetup=false
+    -Dlibiptc=false
+    -Dxkbcommon=false
+    -Dlibcurl=false
+    -Dman=false
+    -Dtests=false
 
     -Ddbuspolicydir=/usr/share/dbus-1/system.d
     -Ddefault-dnssec=no
@@ -183,7 +205,7 @@ build() {
     meson_options+=(-Dgnuefi=true)
   fi
 
-  arch-meson "$pkgbase-stable" build "${meson_options[@]}"
+  arch-meson "$pkgbase-stable" build "${meson_options[@]}" --cross-file cross-file
 
   ninja -C build
 
@@ -198,28 +220,21 @@ build() {
   mv -T "$srcdir/dest"/{systemd,libsystemd}/usr/include/systemd
   mv -T "$srcdir/dest"/{systemd,libsystemd}/usr/lib/pkgconfig/libsystemd.pc
   mv "$srcdir/dest"/systemd/usr/lib/libsystemd.so*      -t "$srcdir/dest"/libsystemd/usr/lib/
-  mv "$srcdir/dest"/systemd/usr/share/man/man3/{SD,sd}* -t "$srcdir/dest"/libsystemd/usr/share/man/man3/
 
   install -dm755 "$srcdir/dest/libudev"/usr/{lib/pkgconfig,share/man/man3,include}
   mv -T "$srcdir/dest"/{systemd,libudev}/usr/include/libudev.h
   mv -T "$srcdir/dest"/{systemd,libudev}/usr/lib/pkgconfig/libudev.pc
   mv "$srcdir/dest"/systemd/usr/lib/libudev.so*       -t "$srcdir/dest"/libudev/usr/lib/
-  mv "$srcdir/dest"/systemd/usr/share/man/man3/*udev* -t "$srcdir/dest"/libudev/usr/share/man/man3/
 
   local nssmodule
   for nssmodule in systemd myhostname mymachines resolve; do
     install -dm755 "$srcdir/dest/nss-$nssmodule"/usr/{lib,share/man/man8}
-    mv -t "$srcdir/dest"/nss-$nssmodule/usr/share/man/man8/ -- \
-       "$srcdir/dest"/systemd/usr/share/man/man8/nss-$nssmodule.8 \
-       "$srcdir/dest"/systemd/usr/share/man/man8/libnss_$nssmodule.*.8
     mv "$srcdir/dest"/systemd/usr/lib/libnss_$nssmodule.so* -t "$srcdir/dest"/nss-$nssmodule/usr/lib/
   done
 
   install -dm755 "$srcdir/dest/systemd-sysvcompat"/usr/share/man/man8
-  mv "$srcdir/dest/systemd"/usr/share/man/man8/{halt,poweroff,reboot,runlevel,shutdown,telinit}.8 \
-     -t "$srcdir/dest/systemd-sysvcompat"/usr/share/man/man8
 
-  rmdir "$srcdir/dest"/systemd/usr/{share/man/man3,lib/pkgconfig,include}
+  rm -rf "$srcdir/dest"/systemd/usr/{share/man/man3,lib/pkgconfig,include}
 }
 
 check() {
@@ -261,7 +276,7 @@ package_systemd() {
   rm -rv "$pkgdir"/etc/systemd/system/*
 
   # we'll create this on installation
-  rmdir "$pkgdir"/var/log/journal/remote
+  rm -rf "$pkgdir"/var/log/journal/remote
 
   # executable (symlinks) shipped with systemd-sysvcompat
   rm "$pkgdir"/usr/bin/{halt,init,poweroff,reboot,runlevel,shutdown,telinit}
