--- a/PKGBUILD	2018-02-21 17:21:51.719593310 +0100
+++ b/PKGBUILD	2018-02-21 17:22:23.987843397 +0100
@@ -17,7 +17,9 @@ md5sums=('abf14cf943f1f8c5e63a24cb26d54f
 
 build() {
     cd unixODBC-${pkgver}
-    ./configure --prefix=/usr --sysconfdir=/etc
+    curl "https://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub;hb=HEAD" \
+      > config.sub
+    ./configure --host=@TARGET@ --build=@BUILDHOST@ --prefix=/usr --sysconfdir=/etc
     make
 }
 
