--- a/PKGBUILD	2018-02-16 15:07:18.597829833 +0100
+++ b/PKGBUILD	2018-02-16 15:06:07.657038447 +0100
@@ -15,7 +15,7 @@ md5sums=('c52f463585d85924b28cdc1e373ae0
 
 build() {
   cd $srcdir/$pkgname-$pkgver
-  ./configure --prefix=/usr --sbindir=/usr/bin --without-python
+  ./configure --host=@TARGET@ --prefix=/usr --sbindir=/usr/bin --without-python
   make 
 }
 
@@ -26,6 +26,7 @@ check() {
 
 package() {
   cd $srcdir/$pkgname-$pkgver
+  export LD_LIBRARY_PATH="@SYSROOT@/lib:$LD_LIBRARY_PATH"
   make DESTDIR=$pkgdir install
   # symlink cracklib-small #34778
   mkdir -p $pkgdir/usr/share/dict
