--- a/PKGBUILD	2018-02-22 14:34:25.657823418 +0100
+++ b/PKGBUILD	2018-02-22 15:01:01.416469952 +0100
@@ -40,6 +40,8 @@ prepare() {
 build() {
   cd $pkgbase
   ./configure --prefix=/usr --sysconfdir=/etc --localstatedir=/var \
+      --host=@TARGET@ --build=@BUILDHOST@ \
+      --disable-silent-rules --without-x --disable-xml-docs \
       --libexecdir=/usr/lib/dbus-1.0 --with-dbus-user=dbus \
       --with-system-pid-file=/run/dbus/pid \
       --with-system-socket=/run/dbus/system_bus_socket \
