--- a/PKGBUILD	2018-02-21 21:52:04.800201713 +0100
+++ b/PKGBUILD	2018-02-21 21:53:10.410318949 +0100
@@ -31,14 +31,15 @@ build() {
   cd $pkgname-$pkgver
 
   ./configure --prefix=/usr \
+    --host=@TARGET@ --build=@BUILDHOST@ \
     --libdir=/usr/lib/libfakeroot \
     --disable-static \
     --with-ipc=sysv
 
   make
 
-  cd doc
-  po4a -k 0 --rm-backups --variable 'srcdir=../doc/' po4a/po4a.cfg
+  #cd doc
+  #po4a -k 0 --rm-backups --variable 'srcdir=../doc/' po4a/po4a.cfg
 }
 
 check() {
