--- a/PKGBUILD	2018-02-21 19:36:19.324934269 +0100
+++ b/PKGBUILD	2018-02-21 19:36:34.145999337 +0100
@@ -19,7 +19,7 @@ prepare() {
 
 build() {
   cd ${pkgname}-${pkgver}
-  ./configure --prefix=/usr --enable-cplusplus --disable-static
+  ./configure --host=@TARGET@ --build=@BUILDHOST@ --prefix=/usr --enable-cplusplus --disable-static
   sed -i -e 's/ -shared / -Wl,-O1,--as-needed\0/g' libtool
   make
 }
