--- a/PKGBUILD	2018-02-21 19:11:52.847881134 +0100
+++ b/PKGBUILD	2018-02-21 19:12:06.357064054 +0100
@@ -42,6 +42,7 @@ validpgpkeys=('4FBD67621082C4C502448E3B1
 build() {
   cd ${pkgname}-${pkgver}
   ./configure --prefix=/usr --libexec=/usr/bin \
+    --host=@TARGET@ --build=@BUILDHOST@ \
     --localstatedir=/var --sysconfdir=/etc \
     --without-wrap --with-pam \
     --enable-ftp --enable-ftpd \
