--- a/PKGBUILD	2018-02-21 16:10:41.962839612 +0100
+++ b/PKGBUILD	2018-02-21 17:05:20.371247141 +0100
@@ -26,6 +26,8 @@ sha1sums=('c98437385d3eaee80c9e2c09f3f0d
 prepare() {
   cd ${pkgbase}-${pkgver}
   patch -p1 -i "${srcdir}"/openldap-ntlm.patch
+  curl "https://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub;hb=HEAD" \
+    > build/config.sub
   sed -i 's|-m 644 $(LIBRARY)|-m 755 $(LIBRARY)|' libraries/{liblber,libldap,libldap_r}/Makefile.in
   sed -i 's|#define LDAPI_SOCK LDAP_RUNDIR LDAP_DIRSEP "run" LDAP_DIRSEP "ldapi"|#define LDAPI_SOCK LDAP_DIRSEP "run" LDAP_DIRSEP "openldap" LDAP_DIRSEP "ldapi"|' include/ldap_defaults.h
   sed -i 's|%LOCALSTATEDIR%/run|/run/openldap|' servers/slapd/slapd.{conf,ldif}
@@ -36,6 +38,8 @@ build() {
   cd ${pkgbase}-${pkgver}
   autoconf
   ./configure --prefix=/usr --libexecdir=/usr/lib \
+    --host=@TARGET@ --build=@BUILDHOST@ CC=@TARGET@-gcc \
+    --with-yielding_select=yes \
     --sysconfdir=/etc --localstatedir=/var/lib/openldap --sbindir=/usr/bin \
     --enable-dynamic --enable-syslog --enable-ipv6 --enable-local \
     --enable-crypt --enable-spasswd --enable-modules \
