--- a/PKGBUILD	2018-02-22 08:32:48.289759692 +0100
+++ b/PKGBUILD	2018-02-22 08:42:21.269426289 +0100
@@ -23,13 +23,16 @@ pkgver() {
 
 prepare() {
   cd $pkgname
-  NOCONFIGURE=1 ./autogen.sh
+  sed -i 's/intltool/#&/; s/gtkdocize/#&/' autogen.sh
+  sed -i '/docs/d' configure.ac
+  NOCONFIGURE=1 ./autogen.sh || true
 }
 
 build() {
   cd $pkgname
   ./configure --prefix=/usr --sysconfdir=/etc --localstatedir=/var \
-    --disable-static --enable-gtk-doc
+    --host=@TARGET@ --build=@BUILDHOST@ \
+    --disable-static --disable-gtk-doc
   make
 }
 
