--- a/PKGBUILD	2018-02-18 21:55:29.272656894 +0100
+++ b/PKGBUILD	2018-02-18 21:57:21.582488355 +0100
@@ -20,9 +20,11 @@ sha256sums=('98fe1d3c311156c597cd5cfa767
 prepare() {
   cd "busybox-$pkgver"
 
-  local safeflags="-march=${CARCH/_/-} -mtune=generic -Os -pipe -fno-strict-aliasing"
+  local safeflags="-Os -pipe -fno-strict-aliasing"
 
   sed 's|^\(CONFIG_EXTRA_CFLAGS\)=.*|\1="'"$safeflags"'"|' "$srcdir/config" > .config
+
+  echo 'CONFIG_CROSS_COMPILER_PREFIX=@TARGET@-' >> .config
 }
 
 build() {
