--- a/PKGBUILD	2018-02-16 19:27:56.577550736 +0100
+++ b/PKGBUILD	2018-02-16 19:29:47.035430721 +0100
@@ -15,7 +15,9 @@ md5sums=('14e7dcd0436d2f49aa403f67e1ef7d
 
 build() {
   cd "$pkgname-$pkgver"
-  ./configure --prefix=/usr --mandir=/usr/share/man
+  curl "https://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub;hb=HEAD" \
+    > config.sub
+  ./configure --host=@TARGET@ --build=@BUILDHOST@ --prefix=/usr --mandir=/usr/share/man
   make
 }
 
