diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-06-07 00:14:42.959731440 +0200
+++ b/PKGBUILD	2018-06-07 07:32:17.853326738 +0200
@@ -10,7 +10,7 @@ pkgrel=1.parabola2
 arch=('i686' 'x86_64' 'armv7h')
 makedepends=('python2-mako' 'libxml2' 'libx11' 'glproto' 'libdrm' 'dri2proto' 'dri3proto' 'presentproto'
              'libxshmfence' 'libxxf86vm' 'libxdamage' 'libvdpau' 'libva' 'wayland' 'wayland-protocols'
-             'elfutils' 'llvm' 'libomxil-bellagio' 'clang' 'libglvnd' 'lm_sensors'
+             'elfutils' 'llvm' 'libomxil-bellagio' 'libglvnd' 'lm_sensors'
              'meson')
 makedepends_x86_64=('libclc' 'libunwind')
 makedepends_i686=("${makedepends_x86_64[@]}")
@@ -76,6 +76,14 @@ build() {
       GALLIUM_XA=false
       LIBUNWIND=false
       ;;
+    powerpc64le)
+      GALLIUM=r300,r600,radeonsi,nouveau,swrast,virgl
+      DRI=r200,radeon,nouveau,swrast
+      VULKAN=
+      LIBUNWIND=
+      VDPAU_AND_XA='--enable-vdpau'
+      OPENCL=
+      ;;
   esac
 
   arch-meson mesa-$pkgver build \
@@ -158,11 +166,6 @@ package_mesa() {
     _install fakeinstall/usr/lib/libwayland*.so*
   fi
 
-  if [ "$CARCH" != "armv7h" ]; then
-    _install fakeinstall/usr/lib/libxatracker.so*
-    _install fakeinstall/usr/lib/libswrAVX*.so*
-  fi
-
   # in libglvnd
   rm -v fakeinstall/usr/lib/libGLESv{1_CM,2}.so*
 
