diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-06-20 07:56:40.877881835 +0200
+++ b/PKGBUILD	2018-06-20 11:06:02.564661867 +0200
@@ -51,6 +51,7 @@ build() {
   cd binutils-build
 
   "$srcdir/binutils-$pkgver/configure" \
+    --host=@CHOST@ --build=@BUILDHOST@ \
     --prefix=/usr \
     --with-lib-path=/usr/lib:/usr/local/lib \
     --with-bugurl=https://bugs.archlinux.org/ \
@@ -61,12 +62,13 @@ build() {
     --enable-plugins \
     --enable-relro \
     --enable-shared \
-    --enable-targets=x86_64-pep \
+    --enable-targets=spu \
+    --enable-targets=powerpc-linux \
     --enable-threads \
     --disable-gdb \
     --disable-werror \
     --with-pic \
-    --with-system-zlib
+    --with-system-zlib=@SYSROOT@/usr/lib/libz.so
 
   make configure-host
   make tooldir=/usr
