diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-05-28 16:39:38.041700185 +0200
+++ b/PKGBUILD	2018-05-28 16:42:42.595035360 +0200
@@ -20,13 +20,15 @@ options=(staticlibs !distcc !ccache)
 source=(https://ftp.gnu.org/gnu/binutils/binutils-$pkgver.tar.xz{,.sig}
         0001-PR22741-objcopy-segfault-on-fuzzed-COFF-object.patch
         0002-PR22829-objcopy-strip-removes-PT_GNU_RELRO-from-lld-.patch
-        0003-PR22836-r-s-doesnt-work-with-g3-using-GCC-7.patch)
+        0003-PR22836-r-s-doesnt-work-with-g3-using-GCC-7.patch
+        https://raw.githubusercontent.com/archlinux32/packages/master/core/binutils/binutils-2.30-branch.patch)
 validpgpkeys=(3A24BC1E8FB409FA9F14371813FCEF89DD9E3C4F)
 md5sums=('ffc476dd46c96f932875d1b2e27e929f'
          'SKIP'
          '469164f3c93a0e92a697537b60c9806c'
          '0c679b37e90fb23de60a4d28329b956a'
-         '53b5682e09c0a27e9994c3efdfe01d29')
+         '53b5682e09c0a27e9994c3efdfe01d29'
+         'a5ba9fc9de327b2eff071e19429c6e67')
 
 prepare() {
   mkdir -p binutils-build
@@ -43,14 +45,14 @@ prepare() {
   # https://sourceware.org/bugzilla/show_bug.cgi?id=22829
   patch -p1 -i "$srcdir/0002-PR22829-objcopy-strip-removes-PT_GNU_RELRO-from-lld-.patch"
 
-  # https://sourceware.org/bugzilla/show_bug.cgi?id=22836
-  patch -p1 -i "$srcdir/0003-PR22836-r-s-doesnt-work-with-g3-using-GCC-7.patch"
+  patch -p1 -i "$srcdir/binutils-2.30-branch.patch"
 }
 
 build() {
   cd binutils-build
 
   "$srcdir/binutils-$pkgver/configure" \
+    --host=@CHOST@ --build=@BUILDHOST@ \
     --prefix=/usr \
     --with-lib-path=/usr/lib:/usr/local/lib \
     --with-bugurl=https://bugs.archlinux.org/ \
@@ -66,7 +68,7 @@ build() {
     --disable-gdb \
     --disable-werror \
     --with-pic \
-    --with-system-zlib
+    --with-system-zlib=@SYSROOT@/usr/lib/libz.so
 
   make configure-host
   make tooldir=/usr
