--- a/PKGBUILD	2018-03-05 11:47:50.860371967 +0100
+++ b/PKGBUILD	2018-03-05 11:47:50.859371959 +0100
@@ -24,7 +24,7 @@ validpgpkeys=('58E0C111E39F5408C5D3EC76C
 
 build() {
   cd $srcdir/$pkgname-$pkgver
-  ./configure --prefix=/usr --with-x --with-appresdir=/usr/share/X11/app-defaults
+  ./configure --host=@CHOST@ --build=@BUILDHOST@ --prefix=/usr --with-x --with-appresdir=/usr/share/X11/app-defaults --with-doc=no
   make
 }
 
