--- a/PKGBUILD	2018-02-20 19:23:38.735472800 +0100
+++ b/PKGBUILD	2018-02-20 19:28:26.237659025 +0100
@@ -24,7 +24,9 @@ md5sums=('3d1147cae66e2959ea5441b183de1b
 build() {
   cd "$pkgname-$pkgver"
 
-  ./configure --prefix=/usr
+  ./configure --host=@CHOST@ --build=@BUILDHOST@ --prefix=/usr \
+    --with-libssl-prefix=@SYSROOT@/usr \
+    LDFLAGS="-L@SYSROOT@/lib -L@SYSROOT@/usr/lib $LDFLAGS"
   make
 }
 
