diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-06-04 06:31:30.220704078 +0200
+++ b/PKGBUILD	2018-06-04 06:35:24.690706413 +0200
@@ -19,7 +19,9 @@ build() {
   # do not use the main makefile, it calls sub make with -e
   # exported CLFAGS by makepkg break the version. see FS#50071
   cd $pkgname
+  CC=@CHOST@-gcc RANLIB=@CHOST@-ranlib AR=@CHOST@-ar \
   make -C lib PREFIX=/usr
+  CC=@CHOST@-gcc RANLIB=@CHOST@-ranlib AR=@CHOST@-ar \
   make -C programs PREFIX=/usr lz4 lz4c
 }
 
@@ -33,6 +35,7 @@ check() {
 
 package() {
   cd $pkgname
+  CC=@CHOST@-gcc RANLIB=@CHOST@-ranlib AR=@CHOST@-ar \
   make install PREFIX=/usr DESTDIR="$pkgdir"
 }
 
