--- a/PKGBUILD	2018-03-09 19:56:31.317644326 +0100
+++ b/PKGBUILD	2018-03-09 19:59:17.896300949 +0100
@@ -34,6 +34,9 @@ fi
 prepare() {
   cd $pkgname-$_basever
 
+  @CONFIG_SUB@ > support/config.sub
+  @CONFIG_GUESS@ > support/config.guess
+
   for (( _p=1; _p<=$((10#${_patchlevel})); _p++ )); do
     msg "applying patch bash${_basever//.}-$(printf "%03d" $_p)"
     patch -p0 -i ../bash${_basever//.}-$(printf "%03d" $_p)
