--- a/PKGBUILD	2018-03-17 15:44:56.987930364 +0100
+++ b/PKGBUILD	2018-03-17 15:46:19.253615593 +0100
@@ -11,7 +11,6 @@ arch=('x86_64')
 url="http://www.cmake.org/"
 license=('custom')
 depends=('curl' 'libarchive' 'shared-mime-info' 'jsoncpp' 'libuv' 'rhash')
-makedepends=('qt5-base' 'python-sphinx' 'emacs')
 optdepends=('qt5-base: cmake-gui'
             'libxkbcommon-x11: cmake-gui')
 source=("https://www.cmake.org/files/v${pkgver%.*}/${pkgname}-${pkgver}.tar.gz"
@@ -30,9 +29,7 @@ build() {
   ./bootstrap --prefix=/usr \
     --mandir=/share/man \
     --docdir=/share/doc/cmake \
-    --sphinx-man \
     --system-libs \
-    --qt-gui \
     --parallel=$(/usr/bin/getconf _NPROCESSORS_ONLN)
   make
 }
@@ -48,12 +45,6 @@ package() {
   ln -s /usr/share/cmake-${pkgver%.*}/editors/vim/syntax/cmake.vim \
     "${vimpath}"/syntax/
 
-  install -d "${pkgdir}"/usr/share/emacs/site-lisp/
-  emacs -batch -f batch-byte-compile \
-    "${pkgdir}"/usr/share/cmake-${pkgver%.*}/editors/emacs/cmake-mode.el
-  ln -s /usr/share/cmake-${pkgver%.*}/editors/emacs/cmake-mode.el \
-    "${pkgdir}"/usr/share/emacs/site-lisp/
-
   install -Dm644 Copyright.txt \
     "${pkgdir}"/usr/share/licenses/${pkgname}/LICENSE
 }
