--- a/PKGBUILD	2018-03-06 09:32:31.395807012 +0100
+++ b/PKGBUILD	2018-03-06 09:35:04.377271050 +0100
@@ -13,6 +13,12 @@ depends=(tcl)
 source=(https://downloads.sourceforge.net/project/expect/Expect/$pkgver/expect${pkgver}.tar.gz)
 sha256sums=('49a7da83b0bdd9f46d04a04deec19c7767bb9a323e40c4781f89caf760b92c34')
 
+prepare() {
+  cd expect${pkgver}
+  @CONFIG_SUB@ > tclconfig/config.sub
+  @CONFIG_GUESS@ > tclconfig/config.guess
+}
+
 build() {
   cd expect${pkgver}
   ./configure --prefix=/usr --mandir=/usr/share/man 
