--- a/PKGBUILD	2018-03-12 18:25:44.581333194 +0100
+++ b/PKGBUILD	2018-03-12 18:27:01.264160919 +0100
@@ -11,7 +11,6 @@ license=('GPL3' 'LGPL2.1')
 url="https://www.gnutls.org/"
 options=('!zipman')
 depends=('gcc-libs' 'libtasn1' 'readline' 'zlib' 'nettle' 'p11-kit' 'libidn' 'libunistring')
-checkdepends=('net-tools')
 optdepends=('guile: for use with Guile bindings')
 source=(https://www.gnupg.org/ftp/gcrypt/gnutls/v3.5/${pkgname}-${pkgver}.tar.xz{,.sig})
 sha256sums=('ae2248d9e78747cf9c469dde81ff8f90b56838b707a0637f3f7d4eee90e80234'
@@ -43,11 +44,6 @@ build() {
   make
 }
 
-check() {
-  cd ${pkgname}-${pkgver}
-  make check
-}
-
 package() {
   cd ${pkgname}-${pkgver}
   make DESTDIR="${pkgdir}" install
