--- a/PKGBUILD	2018-03-06 07:09:51.930237340 +0100
+++ b/PKGBUILD	2018-03-06 07:09:51.930237340 +0100
@@ -14,6 +14,12 @@ depends=('glibc')
 source=('https://github.com/json-c/json-c/archive/json-c-0.13-20171207.tar.gz')
 sha256sums=('26e642456caab38aa9459279b9712ffec52f751e9f46641d28461c244bd6bae6')
 
+prepare() {
+  cd $pkgname-$pkgname-$pkgver-20171207
+  @CONFIG_SUB@ > config.sub
+  @CONFIG_GUESS@ > config.guess
+}
+
 build() {
   cd $pkgname-$pkgname-$pkgver-20171207
   CFLAGS="$CFLAGS -Wno-error"
