--- a/PKGBUILD	2018-03-01 13:46:34.827857959 +0100
+++ b/PKGBUILD	2018-03-01 13:46:34.826857913 +0100
@@ -13,6 +13,12 @@ depends=('glibc')
 source=(http://people.redhat.com/sgrubb/$pkgname/$pkgname-$pkgver.tar.gz)
 md5sums=('0dece96644bd798020e170fbf7663802')
 
+prepare() {
+  cd $pkgname-$pkgver
+  @CONFIG_SUB@ > config.sub
+  @CONFIG_GUESS@ > config.guess
+}
+
 build() {
   cd $pkgname-$pkgver
 
