--- a/PKGBUILD	2018-03-15 19:48:35.405613506 +0100
+++ b/PKGBUILD	2018-03-15 19:50:28.340694183 +0100
@@ -15,6 +15,12 @@ depends=('ncurses')
 source=("http://thrysoee.dk/editline/libedit-${_pkgver}.tar.gz")
 sha256sums=('91f2d90fbd2a048ff6dad7131d9a39e690fd8a8fd982a353f1333dd4017dd4be')
 
+prepare() {
+	cd "${srcdir}/${pkgname}-${_pkgver}"
+  @CONFIG_SUB@ > config.sub
+  @CONFIG_GUESS@ > config.guess
+}
+
 build() {
 	cd "${srcdir}/${pkgname}-${_pkgver}"
 	./configure --prefix=/usr --enable-widec
