--- a/PKGBUILD	2018-03-06 13:37:25.867333943 +0100
+++ b/PKGBUILD	2018-03-06 13:40:46.290388995 +0100
@@ -17,6 +17,12 @@ sha1sums=('a428758999ff573e62d06892e3d2c
           'SKIP')
 validpgpkeys=('D8692123C4065DEA5E0F3AB5249B39D24F25E3B6'  # Werner Koch
               '031EC2536E580D8EA286A9F22071B08A33BD3F06') # NIIBE Yutaka (GnuPG Release Key) <gniibe@fsij.org>
+prepare() {
+  cd ${pkgname}-${pkgver}
+  @CONFIG_SUB@ > build-aux/config.sub
+  @CONFIG_GUESS@ > build-aux/config.guess
+}
+
 build() {
   cd ${pkgname}-${pkgver}
   ./configure --prefix=/usr
