--- a/PKGBUILD	2018-03-11 08:14:01.919722106 +0100
+++ b/PKGBUILD	2018-03-11 08:15:23.920486985 +0100
@@ -10,7 +10,6 @@ pkgdesc='JPEG image codec with accelerat
 url='https://libjpeg-turbo.org/'
 arch=('x86_64')
 license=('custom')
-makedepends=('nasm' 'jdk8-openjdk')
 validpgpkeys=('7D6293CC6378786E1B5C496885C7044E033FDE16')
 source=("https://downloads.sourceforge.net/project/${pkgname}/${pkgver}/${pkgname}-${pkgver}.tar.gz"{,.sig})
 sha256sums=('b24890e2bb46e12e72a79f7e965f409f4e16466d00e1dd15d93d73ee6b592523'
@@ -36,7 +37,7 @@ build() {
 		--prefix=/usr \
 		--mandir=/usr/share/man \
 		--with-jpeg8 \
-		--with-java \
+		--without-java \
 		--with-build-date=$_build \
 
 	make
