--- a/PKGBUILD	2018-03-06 16:22:20.333070800 +0100
+++ b/PKGBUILD	2018-03-06 16:26:07.269268960 +0100
@@ -14,6 +14,12 @@ source=(https://www.gnupg.org/ftp/gcrypt
 sha1sums=('a98385734a0c3f5b713198e8d6e6e4aeb0b76fde'
           'SKIP')
 validpgpkeys=('D8692123C4065DEA5E0F3AB5249B39D24F25E3B6') # Werner Koch
+prepare() {
+  cd $srcdir/$pkgname-$pkgver
+  @CONFIG_SUB@ > build-aux/config.sub
+  @CONFIG_GUESS@ > build-aux/config.guess
+}
+
 build() {
   cd $srcdir/$pkgname-$pkgver
   ./configure --prefix=/usr
