--- a/PKGBUILD	2018-03-06 09:59:02.123112851 +0100
+++ b/PKGBUILD	2018-03-06 10:06:27.202915639 +0100
@@ -9,7 +9,7 @@ url='https://github.com/rockdaboot/libps
 arch=('x86_64')
 license=('MIT')
 depends=('libidn2' 'libunistring')
-makedepends=('libxslt' 'python' 'publicsuffix-list' 'gtk-doc')
+makedepends=('libxslt' 'publicsuffix-list')
 source=(https://github.com/rockdaboot/libpsl/releases/download/${pkgname}-${pkgver}/${pkgname}-${pkgver}.tar.gz)
 sha512sums=('2a88024536cb89ad96828f9f94c21af4bd4136fc19d16e0769762e196db4b38969cd54d0877a959f64fbb96b1533e0d59740c40a664e2ea27df3266ae8ab0477')
 
@@ -26,7 +26,7 @@ build() {
     --disable-static \
     --disable-dependency-tracking \
     --enable-man \
-    --enable-gtk-doc \
+    --disable-gtk-doc \
     --enable-{builtin,runtime}=libidn2 \
     --with-psl-file=/usr/share/publicsuffix/effective_tld_names.dat \
     --with-psl-testfile=/usr/share/publicsuffix/test_psl.txt
