--- a/PKGBUILD	2018-03-15 16:43:18.938199115 +0100
+++ b/PKGBUILD	2018-03-15 17:00:27.931732044 +0100
@@ -9,8 +9,7 @@ arch=(x86_64)
 license=(LGPL)
 url="https://wiki.gnome.org/Projects/Libsecret"
 depends=(glib2 libgcrypt)
-checkdepends=(python-dbus dbus-glib python-gobject gjs)
-makedepends=(intltool docbook-xsl gobject-introspection vala git gtk-doc)
+makedepends=(intltool gobject-introspection git gtk-doc)
 optdepends=('gnome-keyring: key storage service (or use any other service implementing org.freedesktop.secrets)')
 _commit=998065599c66055dcffa1ef1ddebb947ccd68248  # master
 source=("git+https://git.gnome.org/browse/libsecret#commit=$_commit")
@@ -29,15 +28,10 @@ prepare() {
 build() {
   cd $pkgname
   ./configure --prefix=/usr --sysconfdir=/etc --localstatedir=/var \
-    --disable-static --enable-gtk-doc
+    --disable-static --disable-gtk-doc
   make
 }
 
-check() {
-  cd $pkgname
-  dbus-run-session make -k check 
-}
-
 package() {
   cd $pkgname
   make DESTDIR="$pkgdir" install
