--- a/PKGBUILD	2018-03-18 19:02:12.204520735 +0100
+++ b/PKGBUILD	2018-03-18 19:09:40.170930659 +0100
@@ -14,7 +14,7 @@ arch=('x86_64')
 url="http://cyrusimap.web.cmu.edu/"
 license=('custom')
 options=('!makeflags')
-makedepends=('postgresql-libs' 'libmariadbclient' 'libldap' 'krb5' 'openssl' 'sqlite')
+makedepends=('libldap' 'krb5' 'openssl' 'sqlite')
 source=(ftp://ftp.cyrusimap.org/cyrus-sasl/cyrus-sasl-${pkgver}.tar.gz
         cyrus-sasl-2.1.22-qa.patch
         cyrus-sasl-2.1.26-size_t.patch
@@ -111,9 +111,6 @@ build() {
       --enable-login \
       --enable-ntlm \
       --disable-passdss \
-      --enable-sql \
-      --with-mysql=/usr \
-      --with-pgsql=/usr/lib \
       --with-sqlite3=/usr/lib \
       --enable-ldapdb \
       --disable-macos-framework \
