--- a/PKGBUILD	2018-03-06 13:35:45.556300299 +0100
+++ b/PKGBUILD	2018-03-06 13:36:38.606847448 +0100
@@ -8,14 +8,14 @@
 pkgname=libtool
 pkgver=2.4.6+40+g6ca5e224
 pkgrel=4
-_gccver=7.3.0
+_gccver=7.3.1
 pkgdesc='A generic library support script'
 arch=(x86_64)
 url='http://www.gnu.org/software/libtool'
 license=(GPL)
 groups=(base-devel)
 depends=(sh tar glibc)
-makedepends=("gcc=$_gccver" git help2man)
+makedepends=("gcc=$_gccver" git)
 provides=("libltdl=$pkgver" "libtool-multilib=$pkgver")
 conflicts=('libltdl' 'libtool-multilib')
 replaces=('libltdl' 'libtool-multilib')
