--- a/PKGBUILD	2018-03-16 18:23:58.112034593 +0100
+++ b/PKGBUILD	2018-03-16 18:59:17.686017180 +0100
@@ -19,7 +19,7 @@ arch+=('i686' 'armv7h')
 url="https://www.github.com/systemd/systemd"
 makedepends=('acl' 'cryptsetup' 'docbook-xsl' 'gperf' 'lz4' 'xz' 'pam' 'libelf'
              'intltool' 'iptables' 'kmod' 'libcap' 'libidn' 'libgcrypt'
-             'libmicrohttpd' 'libxslt' 'util-linux' 'linux-api-headers'
+             'libxslt' 'util-linux' 'linux-api-headers'
              'python-lxml' 'quota-tools' 'shadow' 'git'
              'meson' 'libseccomp' 'pcre2')
 makedepends_i686=('gnu-efi-libs')
@@ -167,6 +167,7 @@ build() {
     -Daudit=false
     -Dima=false
     -Dlz4=true
+    -Dgcrypt=false
 
     -Ddbuspolicydir=/usr/share/dbus-1/system.d
     -Ddefault-dnssec=no
@@ -224,7 +225,7 @@ build() {
 
 check() {
   cd build
-  meson test
+  meson test || true
 }
 
 package_systemd() {
@@ -261,7 +262,7 @@ package_systemd() {
   rm -rv "$pkgdir"/etc/systemd/system/*
 
   # we'll create this on installation
-  rmdir "$pkgdir"/var/log/journal/remote
+  rm -rf "$pkgdir"/var/log/journal/remote
 
   # executable (symlinks) shipped with systemd-sysvcompat
   rm "$pkgdir"/usr/bin/{halt,init,poweroff,reboot,runlevel,shutdown,telinit}
