--- a/PKGBUILD	2018-03-01 13:40:15.003223269 +0100
+++ b/PKGBUILD	2018-03-01 13:40:15.003223269 +0100
@@ -13,6 +13,12 @@ options=('staticlibs')
 source=(http://rpm5.org/files/${pkgname}/${pkgname}-${pkgver}.tar.gz)
 sha1sums=('cfe94a15a2404db85858a81ff8de27c8ff3e235e')
 
+prepare() {
+  cd "${srcdir}/${pkgname}-${pkgver}"
+  @CONFIG_SUB@ > config.sub
+  @CONFIG_GUESS@ > config.guess
+}
+
 build() {
   cd "${srcdir}/${pkgname}-${pkgver}"
   ./configure --prefix=/usr
