--- a/PKGBUILD	2018-03-16 16:30:38.465266947 +0100
+++ b/PKGBUILD	2018-03-16 16:38:53.114207732 +0100
@@ -15,6 +15,12 @@ options=('!emptydirs')
 source=(http://downloads.sourceforge.net/sourceforge/linuxquota/quota-${pkgver}.tar.gz)
 sha1sums=('38e6cc7538d504891a8454bbdabef058bd1adc11')
 
+prepare() {
+  cd quota-$pkgver
+  @CONFIG_SUB@ > config.sub
+  @CONFIG_GUESS@ > config.guess
+}
+
 build() {
   cd quota-$pkgver
   ./configure --prefix=/usr --sysconfdir=/etc
