--- a/PKGBUILD	2018-03-16 21:28:58.113879932 +0100
+++ b/PKGBUILD	2018-03-16 21:35:47.180524060 +0100
@@ -11,7 +11,7 @@ url='https://github.com/shadow-maint/sha
 license=('BSD')
 groups=('base')
 depends=('bash' 'pam' 'acl')
-makedepends=('git' 'libxslt' 'docbook-xsl' 'gnome-doc-utils')
+makedepends=('git' 'libxslt' 'docbook-xsl' 'gnome-doc-utils' 'python2')
 backup=(etc/login.defs
         etc/pam.d/{chage,passwd,shadow,useradd,usermod,userdel}
         etc/pam.d/{chpasswd,newusers,groupadd,groupdel,groupmod}
