--- a/PKGBUILD	2018-03-03 22:03:14.363545840 +0100
+++ b/PKGBUILD	2018-03-03 22:03:14.363545840 +0100
@@ -12,22 +12,23 @@ url='http://sourceware.org/libffi/'
 license=('MIT')
 depends=('glibc')
 checkdepends=('dejagnu')
-source=(ftp://sourceware.org/pub/libffi/libffi-$pkgver.tar.gz)
-sha1sums=('280c265b789e041c02e5c97815793dfc283fb1e6')
+source=("git+https://github.com/sorear/libffi-riscv.git#branch=riscv-$pkgver")
+sha1sums=('SKIP')
 
 build() {
-  cd $pkgname-$pkgver
-  ./configure --prefix=/usr --disable-static \
+  cd libffi-riscv
+  autoreconf --install
+  ./configure --host=@CHOST@ --build=@BUILDHOST@ --prefix=/usr --disable-static \
     --enable-pax_emutramp
   make
 }
 
 check() {
-  make -C $pkgname-$pkgver check
+  make -C libffi-riscv check
 }
 
 package() {
-  cd $pkgname-$pkgver
+  cd libffi-riscv
   make DESTDIR="$pkgdir" install
   install -Dm644 LICENSE "$pkgdir"/usr/share/licenses/$pkgname/LICENSE
 }
